<!DOCTYPE html>
<html lang="en">
<head>
<meta content="text/html; charset=utf-8" http-equiv="content-type"/>
<title>Analyzing Tumors by Synthesis</title>
<!--Generated on Mon Sep  9 19:51:32 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport"/>
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/ar5iv.0.7.9.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/ar5iv-fonts.0.7.9.min.css" rel="stylesheet" type="text/css"/>
<link href="/static/browse/0.3.4/css/latexml_styles.css" rel="stylesheet" type="text/css"/>
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/html2canvas/1.3.3/html2canvas.min.js"></script>
<script src="/static/browse/0.3.4/js/addons_new.js"></script>
<script src="/static/browse/0.3.4/js/feedbackOverlay.js"></script>
<base href="/html/2409.06035v1/"/></head>
<body>
<nav class="ltx_page_navbar">
<nav class="ltx_TOC">
<ol class="ltx_toclist">
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S1" title="In Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1 </span>Introduction</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S1.SS1" title="In 1 Introduction ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1.1 </span>Why Synthetic Data?</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S1.SS2" title="In 1 Introduction ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">1.2 </span>Real vs. Synthetic Data</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S2" title="In Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2 </span>Detecting Real Tumors in CT Scans</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S2.SS1" title="In 2 Detecting Real Tumors in CT Scans ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.1 </span>Tumors in Solid Organs</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S2.SS2" title="In 2 Detecting Real Tumors in CT Scans ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.2 </span>Tumors in Tubular Organs</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S2.SS3" title="In 2 Detecting Real Tumors in CT Scans ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">2.3 </span>High Similarity in Early-Stage Tumors</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S3" title="In Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3 </span>Technical Barriers and Clinical Needs</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S3.SS1" title="In 3 Technical Barriers and Clinical Needs ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.1 </span>Technical Barriers</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S3.SS2" title="In 3 Technical Barriers and Clinical Needs ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">3.2 </span>Clinical Needs</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S4" title="In Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4 </span>Technology Trend I: Modeling-based Approaches</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S4.SS1" title="In 4 Technology Trend I: Modeling-based Approaches ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.1 </span>Property of Pixel2Cancer</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S4.SS2" title="In 4 Technology Trend I: Modeling-based Approaches ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">4.2 </span>Clinical Perspectives</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S5" title="In Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5 </span>Technology Trend II: Learning-based Approaches</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S5.SS1" title="In 5 Technology Trend II: Learning-based Approaches ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5.1 </span>Property of DiffTumor</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S5.SS2" title="In 5 Technology Trend II: Learning-based Approaches ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">5.2 </span>Clinical Perspectives</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section">
<a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S6" title="In Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">6 </span>Tumor Synthesis Benchmark</span></a>
<ol class="ltx_toclist ltx_toclist_section">
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S6.SS1" title="In 6 Tumor Synthesis Benchmark ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">6.1 </span>Case Study: Fake Tumors, Real Results</span></a></li>
<li class="ltx_tocentry ltx_tocentry_subsection"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S6.SS2" title="In 6 Tumor Synthesis Benchmark ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">6.2 </span>Visual Turing Test</span></a></li>
</ol>
</li>
<li class="ltx_tocentry ltx_tocentry_section"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.S7" title="In Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_title"><span class="ltx_tag ltx_tag_ref">7 </span>Conclusion</span></a></li>
</ol></nav>
</nav>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line"><span class="ltx_note ltx_role_institutetext" id="id1"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup><span class="ltx_note_type">institutetext: </span>Qi Chen </span></span></span><span class="ltx_note ltx_role_institutetext" id="id2"><sup class="ltx_note_mark">2</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">2</sup><span class="ltx_note_type">institutetext: </span>University of Chinese Academy of Sciences </span></span></span><span class="ltx_note ltx_role_institutetext" id="id3"><sup class="ltx_note_mark">3</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">3</sup><span class="ltx_note_type">institutetext: </span><span class="ltx_note ltx_role_email" id="id3.1"><sup class="ltx_note_mark">3</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">3</sup><span class="ltx_note_type">email: </span>chenqi24@ucas.ac.cn</span></span></span>
</span></span></span><span class="ltx_note ltx_role_institutetext" id="id4"><sup class="ltx_note_mark">4</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">4</sup><span class="ltx_note_type">institutetext: </span>Yuxiang Lai </span></span></span><span class="ltx_note ltx_role_institutetext" id="id5"><sup class="ltx_note_mark">5</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">5</sup><span class="ltx_note_type">institutetext: </span>Emory University </span></span></span><span class="ltx_note ltx_role_institutetext" id="id6"><sup class="ltx_note_mark">6</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">6</sup><span class="ltx_note_type">institutetext: </span><span class="ltx_note ltx_role_email" id="id6.1"><sup class="ltx_note_mark">6</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">6</sup><span class="ltx_note_type">email: </span>ylai76@emory.edu</span></span></span>
</span></span></span><span class="ltx_note ltx_role_institutetext" id="id7"><sup class="ltx_note_mark">7</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">7</sup><span class="ltx_note_type">institutetext: </span>Xiaoxi Chen </span></span></span><span class="ltx_note ltx_role_institutetext" id="id8"><sup class="ltx_note_mark">8</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">8</sup><span class="ltx_note_type">institutetext: </span>University of Illinois Urbana-Champaign </span></span></span><span class="ltx_note ltx_role_institutetext" id="id9"><sup class="ltx_note_mark">9</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">9</sup><span class="ltx_note_type">institutetext: </span><span class="ltx_note ltx_role_email" id="id9.1"><sup class="ltx_note_mark">9</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">9</sup><span class="ltx_note_type">email: </span>xiaoxic3@illinois.edu</span></span></span>
</span></span></span><span class="ltx_note ltx_role_institutetext" id="id10"><sup class="ltx_note_mark">10</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">10</sup><span class="ltx_note_type">institutetext: </span>Qixin Hu </span></span></span><span class="ltx_note ltx_role_institutetext" id="id11"><sup class="ltx_note_mark">11</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">11</sup><span class="ltx_note_type">institutetext: </span>University of Southern California </span></span></span><span class="ltx_note ltx_role_institutetext" id="id12"><sup class="ltx_note_mark">12</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">12</sup><span class="ltx_note_type">institutetext: </span><span class="ltx_note ltx_role_email" id="id12.1"><sup class="ltx_note_mark">12</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">12</sup><span class="ltx_note_type">email: </span>qixinhu@usc.edu</span></span></span>
</span></span></span><span class="ltx_note ltx_role_institutetext" id="id13"><sup class="ltx_note_mark">13</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">13</sup><span class="ltx_note_type">institutetext: </span>Alan Yuille </span></span></span><span class="ltx_note ltx_role_institutetext" id="id14"><sup class="ltx_note_mark">14</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">14</sup><span class="ltx_note_type">institutetext: </span>Johns Hopkins University </span></span></span><span class="ltx_note ltx_role_institutetext" id="id15"><sup class="ltx_note_mark">15</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">15</sup><span class="ltx_note_type">institutetext: </span><span class="ltx_note ltx_role_email" id="id15.1"><sup class="ltx_note_mark">15</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">15</sup><span class="ltx_note_type">email: </span>ayuille1@jhu.edu</span></span></span>
</span></span></span><span class="ltx_note ltx_role_institutetext" id="id16"><sup class="ltx_note_mark">16</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">16</sup><span class="ltx_note_type">institutetext: </span>Zongwei Zhou (🖂) </span></span></span><span class="ltx_note ltx_role_institutetext" id="id17"><sup class="ltx_note_mark">17</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">17</sup><span class="ltx_note_type">institutetext: </span>Johns Hopkins University </span></span></span><span class="ltx_note ltx_role_institutetext" id="id18"><sup class="ltx_note_mark">18</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">18</sup><span class="ltx_note_type">institutetext: </span><span class="ltx_note ltx_role_email" id="id18.1"><sup class="ltx_note_mark">18</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">18</sup><span class="ltx_note_type">email: </span>zzhou82@jh.edu</span></span></span>
</span></span></span>
<h1 class="ltx_title ltx_title_document">Analyzing Tumors by Synthesis</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Qi Chen
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Yuxiang Lai
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Xiaoxi Chen
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Qixin Hu
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Alan Yuille and Zongwei Zhou
</span></span>
</div>
<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<p class="ltx_p" id="id1.id1">Computer-aided tumor detection has shown great potential in enhancing the interpretation of over 80 million CT scans performed annually in the United States. However, challenges arise due to the rarity of CT scans with tumors, especially early-stage tumors. Developing AI with real tumor data faces issues of scarcity, annotation difficulty, and low prevalence. Tumor synthesis addresses these challenges by generating numerous tumor examples in medical images, aiding AI training for tumor detection and segmentation. Successful synthesis requires realistic and generalizable synthetic tumors across various organs. This chapter reviews AI development on real and synthetic data and summarizes two key trends in synthetic data for cancer imaging research: modeling-based and learning-based approaches. Modeling-based methods, like Pixel2Cancer, simulate tumor development over time using generic rules, while learning-based methods, like DiffTumor, learn from a few annotated examples in one organ to generate synthetic tumors in others. Reader studies with expert radiologists show that synthetic tumors can be convincingly realistic. We also present case studies in the liver, pancreas, and kidneys reveal that AI trained on synthetic tumors can achieve performance comparable to, or better than, AI only trained on real data. Tumor synthesis holds significant promise for expanding datasets, enhancing AI reliability, improving tumor detection performance, and preserving patient privacy.</p>
</div>
<section class="ltx_section" id="Ch0.S1">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>
<div class="ltx_para" id="Ch0.S1.p1">
<p class="ltx_p" id="Ch0.S1.p1.1">Medical image analysis aims to derive detailed information non-invasively about a patient’s medical condition, including the disease’s origin, precise location, and its relationship with adjacent tissues. Specifically, for symptoms that cannot be directly diagnosed, medical professionals employ various imaging devices to capture detailed images of target organs for disease screening, diagnosis, and treatment. Therefore, medical imaging systems generate vast amounts of medical image data daily. This data may encompass different organs of the body, as well as tissues and pathological regions associated with diseases.</p>
</div>
<div class="ltx_para" id="Ch0.S1.p2">
<p class="ltx_p" id="Ch0.S1.p2.1">Medical images come from modalities like X-ray, CT, MRI, and PET. This data, referred to as <span class="ltx_text ltx_font_bold" id="Ch0.S1.p2.1.1">real data</span> in this chapter, can be analyzed using post-processing and artificial intelligence (AI) to reveal details not visible to the naked eye, aiding in disease detection such as detecting tumors at their early stage. However, managing real-world data for AI-driven diagnostics is challenging. <span class="ltx_text ltx_font_bold" id="Ch0.S1.p2.1.2">Synthetic data</span> offers a promising alternative, potentially allowing AI to generalize better to real-world scenarios and overcome the difficulties of using real data for training. Generally, synthetic data refer to artificially generated data that mimic the characteristics and structure of real data without being directly derived from actual observations (Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F1" title="Figure 1 ‣ 1.1 Why Synthetic Data? ‣ 1 Introduction ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">1</span></a>).</p>
</div>
<section class="ltx_subsection" id="Ch0.S1.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.1 </span>Why Synthetic Data?</h3>
<div class="ltx_para" id="Ch0.S1.SS1.p1">
<p class="ltx_p" id="Ch0.S1.SS1.p1.1">Synthetic data are vital in AI research due to the challenges of acquiring real data, including time constraints, high costs, patient privacy concerns, and manual effort <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib122" title="">zhou2019models </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib24" title="">feng2020parts2whole </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib31" title="">haghighi2020learning </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib32" title="">haghighi2021transferable </a></cite>. They provide significant advantages by saving time and reducing the need for extensive manual annotation. The use of AI-generated content (AIGC) has proven effective across various domains, including medical imaging, where it serves both as a training resource for AI models and as a means of evaluating their performance with realistic yet hard-to-obtain data (see Table <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.T1" title="Table 1 ‣ 1.1 Why Synthetic Data? ‣ 1 Introduction ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">1</span></a>). Synthetic data offer precise control over properties such as shape, texture, and location, which is particularly valuable in medical applications. This control enables the creation of diverse and representative datasets for model training and provides useful examples for medical education and patient communication. Additionally, controllable synthetic tumors facilitate AI debugging and model diagnostics, enhancing the interpretability of AI behavior. Increasing evidence supports that synthetic data can improve AI performance, making it a powerful tool for advancing research and improving outcomes in fields like oncology.</p>
</div>
<figure class="ltx_figure" id="Ch0.F1"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="474" id="Ch0.F1.g1" src="extracted/5843742/fig_real_synthetic_examples.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 1: </span><span class="ltx_text ltx_font_italic" id="Ch0.F1.2.1">Can you distinguish synthetic data from real data in different modalities?</span> (a) X-ray image examples <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib26" title="">gao2023synthetic </a></cite>. (b) CT image examples <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib33" title="">hamamci2023generatect </a></cite>. (c) MR image examples <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib77" title="">park2021generative </a></cite>. (d) PET image example. (e) Endoscopy image example <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib64" title="">li2024endora </a></cite>. (f) Histopathology image example <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib4" title="">aversa2024diffinfinite </a></cite>.</figcaption>
</figure>
<figure class="ltx_table" id="Ch0.T1">
<figcaption class="ltx_caption" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 1: </span>
A summary of existing synthesis methods for medical imaging encompasses various aspects: body part, disease type, imaging modality, dataset utilized, and generative model type. Additionally, we are maintaining a <a class="ltx_ref ltx_href" href="https://github.com/MrGiovanni/SyntheticTumors/blob/main/AWESOME.md" title="">webpage</a> to track the latest publications and corresponding code repositories on data synthesis in medicine more comprehensively.
</figcaption><div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<table class="ltx_tabular ltx_figure_panel ltx_align_middle" id="Ch0.T1.4">
<tr class="ltx_tr" id="Ch0.T1.4.1">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T1.4.1.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.1.1.1">
<span class="ltx_p" id="Ch0.T1.4.1.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.1.1.1.1.1" style="font-size:70%;">reference</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T1.4.1.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.1.2.1">
<span class="ltx_p" id="Ch0.T1.4.1.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.1.2.1.1.1" style="font-size:70%;">body part</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T1.4.1.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.1.3.1">
<span class="ltx_p" id="Ch0.T1.4.1.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.1.3.1.1.1" style="font-size:70%;">disease</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T1.4.1.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.1.4.1">
<span class="ltx_p" id="Ch0.T1.4.1.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.1.4.1.1.1" style="font-size:70%;">modality</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T1.4.1.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.1.5.1">
<span class="ltx_p" id="Ch0.T1.4.1.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.1.5.1.1.1" style="font-size:70%;">dataset</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T1.4.1.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.1.6.1">
<span class="ltx_p" id="Ch0.T1.4.1.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.1.6.1.1.1" style="font-size:70%;">generative model</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.2">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T1.4.2.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.2.1.1">
<span class="ltx_p" id="Ch0.T1.4.2.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.2.1.1.1.1" style="font-size:70%;">Teixeira et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib97" title="">teixeira2018generating </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T1.4.2.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.2.2.1">
<span class="ltx_p" id="Ch0.T1.4.2.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.2.2.1.1.1" style="font-size:70%;">whole body</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T1.4.2.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.2.3.1">
<span class="ltx_p" id="Ch0.T1.4.2.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.2.3.1.1.1" style="font-size:70%;">anomaly detection</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T1.4.2.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.2.4.1">
<span class="ltx_p" id="Ch0.T1.4.2.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.2.4.1.1.1" style="font-size:70%;">X-ray</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T1.4.2.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.2.5.1">
<span class="ltx_p" id="Ch0.T1.4.2.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.2.5.1.1.1" style="font-size:70%;">private dataset</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T1.4.2.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.2.6.1">
<span class="ltx_p" id="Ch0.T1.4.2.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.2.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.3" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.3.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.3.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.3.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.3.1.1.1.1" style="font-size:70%;">Wu et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib107" title="">wu2018conditional </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.3.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.3.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.3.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.3.2.1.1.1" style="font-size:70%;">chest</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.3.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.3.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.3.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.3.3.1.1.1" style="font-size:70%;">breast cancer</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.3.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.3.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.3.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.3.4.1.1.1" style="font-size:70%;">X-ray</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.3.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.3.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.3.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.3.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.3.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.3.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.3.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.3.5.1.1.2.1.1.1">DDSM <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib37" title="">heath1998current </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.3.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.3.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.3.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.3.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.3.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.3.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.4">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.4.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.4.1.1">
<span class="ltx_p" id="Ch0.T1.4.4.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.4.1.1.1.1" style="font-size:70%;">Gao et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib26" title="">gao2023synthetic </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.4.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.4.2.1">
<span class="ltx_p" id="Ch0.T1.4.4.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.4.2.1.1.1" style="font-size:70%;">bone &amp; chest</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.4.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.4.3.1">
<span class="ltx_p" id="Ch0.T1.4.4.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.4.3.1.1.1" style="font-size:70%;">COVID-19 lesion</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.4.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.4.4.1">
<span class="ltx_p" id="Ch0.T1.4.4.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.4.4.1.1.1" style="font-size:70%;">X-ray</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.4.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.4.5.1">
<span class="ltx_p" id="Ch0.T1.4.4.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.4.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.4.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.4.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.4.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.4.5.1.1.2.1.1.1">COVID-19 CXR <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib101" title="">waheed2020covidgan </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.4.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.4.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.4.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.4.6.1">
<span class="ltx_p" id="Ch0.T1.4.4.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.4.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.5" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.5.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.5.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.5.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.5.1.1.1.1" style="font-size:70%;">Jin et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib48" title="">jin2018ct </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.5.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.5.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.5.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.5.2.1.1.1" style="font-size:70%;">chest</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.5.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.5.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.5.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.5.3.1.1.1" style="font-size:70%;">lung nodule</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.5.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.5.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.5.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.5.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.5.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.5.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.5.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.5.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.5.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.5.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.5.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.5.5.1.1.2.1.1.1">LIDC <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib3" title="">armato2011lung </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.5.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.5.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.5.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.5.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.5.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.5.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.6">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.6.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.6.1.1">
<span class="ltx_p" id="Ch0.T1.4.6.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.6.1.1.1.1" style="font-size:70%;">Yao et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib113" title="">yao2021label </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.6.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.6.2.1">
<span class="ltx_p" id="Ch0.T1.4.6.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.6.2.1.1.1" style="font-size:70%;">chest</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.6.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.6.3.1">
<span class="ltx_p" id="Ch0.T1.4.6.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.6.3.1.1.1" style="font-size:70%;">COVID-19</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.6.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.6.4.1">
<span class="ltx_p" id="Ch0.T1.4.6.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.6.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.6.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.6.5.1">
<span class="ltx_p" id="Ch0.T1.4.6.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.6.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.6.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.6.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.6.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.6.5.1.1.2.1.1.1">LUNA16 <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib87" title="">setio2017validation </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.6.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.6.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.6.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.6.6.1">
<span class="ltx_p" id="Ch0.T1.4.6.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.6.6.1.1.1" style="font-size:70%;">hand-crafted designs</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.7" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.7.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.7.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.7.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.7.1.1.1.1" style="font-size:70%;">Jiang et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib47" title="">jiang2020covid </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.7.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.7.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.7.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.7.2.1.1.1" style="font-size:70%;">chest</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.7.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.7.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.7.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.7.3.1.1.1" style="font-size:70%;">Covid-19</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.7.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.7.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.7.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.7.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.7.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.7.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.7.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.7.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.7.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.7.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.7.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.7.5.1.1.2.1.1.1">COVID <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib112" title="">yang2020covid </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.7.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.7.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.7.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.7.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.7.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.7.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.8">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.8.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.8.1.1">
<span class="ltx_p" id="Ch0.T1.4.8.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.8.1.1.1.1" style="font-size:70%;">Jin et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib49" title="">jin2021free </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.8.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.8.2.1">
<span class="ltx_p" id="Ch0.T1.4.8.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.8.2.1.1.1" style="font-size:70%;">abdomen</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.8.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.8.3.1">
<span class="ltx_p" id="Ch0.T1.4.8.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.8.3.1.1.1" style="font-size:70%;">liver &amp; kidney tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.8.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.8.4.1">
<span class="ltx_p" id="Ch0.T1.4.8.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.8.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.8.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.8.5.1">
<span class="ltx_p" id="Ch0.T1.4.8.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.8.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.8.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.8.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.8.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.8.5.1.1.2.1.1.1">LiTS <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib8" title="">bilic2023liver </a></cite> &amp; KiTS <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib38" title="">heller2021state </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.8.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.8.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.8.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.8.6.1">
<span class="ltx_p" id="Ch0.T1.4.8.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.8.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.9" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.9.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.9.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.9.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.9.1.1.1.1" style="font-size:70%;">Wei et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib105" title="">wei2022pancreatic </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.9.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.9.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.9.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.9.2.1.1.1" style="font-size:70%;">abdomen</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.9.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.9.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.9.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.9.3.1.1.1" style="font-size:70%;">pancreatic tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.9.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.9.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.9.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.9.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.9.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.9.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.9.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.9.5.1.1.1" style="font-size:70%;">private dataset</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.9.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.9.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.9.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.9.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.10">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.10.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.10.1.1">
<span class="ltx_p" id="Ch0.T1.4.10.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.10.1.1.1.1" style="font-size:70%;">Lyu et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib70" title="">lyu2022learning </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.10.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.10.2.1">
<span class="ltx_p" id="Ch0.T1.4.10.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.10.2.1.1.1" style="font-size:70%;">abdomen</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.10.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.10.3.1">
<span class="ltx_p" id="Ch0.T1.4.10.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.10.3.1.1.1" style="font-size:70%;">liver tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.10.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.10.4.1">
<span class="ltx_p" id="Ch0.T1.4.10.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.10.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.10.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.10.5.1">
<span class="ltx_p" id="Ch0.T1.4.10.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.10.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.10.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.10.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.10.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.10.5.1.1.2.1.1.1">LiTS <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib8" title="">bilic2023liver </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.10.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.10.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.10.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.10.6.1">
<span class="ltx_p" id="Ch0.T1.4.10.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.10.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.11" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.11.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.11.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.11.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.11.1.1.1.1" style="font-size:70%;">Hu et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib42" title="">hu2023label </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.11.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.11.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.11.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.11.2.1.1.1" style="font-size:70%;">abdomen</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.11.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.11.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.11.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.11.3.1.1.1" style="font-size:70%;">liver tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.11.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.11.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.11.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.11.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.11.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.11.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.11.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.11.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.11.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.11.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.11.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.11.5.1.1.2.1.1.1">LiTS <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib8" title="">bilic2023liver </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.11.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.11.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.11.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.11.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.11.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.11.6.1.1.1" style="font-size:70%;">hand-crafted designs</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.12">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.12.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.12.1.1">
<span class="ltx_p" id="Ch0.T1.4.12.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.12.1.1.1.1" style="font-size:70%;">Li et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib63" title="">li2023early </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.12.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.12.2.1">
<span class="ltx_p" id="Ch0.T1.4.12.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.12.2.1.1.1" style="font-size:70%;">abdomen</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.12.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.12.3.1">
<span class="ltx_p" id="Ch0.T1.4.12.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.12.3.1.1.1" style="font-size:70%;">pancreatic tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.12.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.12.4.1">
<span class="ltx_p" id="Ch0.T1.4.12.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.12.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.12.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.12.5.1">
<span class="ltx_p" id="Ch0.T1.4.12.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.12.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.12.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.12.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.12.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.12.5.1.1.2.1.1.1">MSD <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib2" title="">antonelli2022medical </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.12.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.12.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.12.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.12.6.1">
<span class="ltx_p" id="Ch0.T1.4.12.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.12.6.1.1.1" style="font-size:70%;">hand-crafted designs</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.13" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.13.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.13.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.13.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.13.1.1.1.1" style="font-size:70%;">Chen et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib9" title="">chen2024towards </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.13.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.13.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.13.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.13.2.1.1.1" style="font-size:70%;">abdomen</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.13.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.13.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.13.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.13.3.1.1.1" style="font-size:70%;">tumors in liver, pancreas and kidney</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.13.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.13.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.13.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.13.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.13.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.13.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.13.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.13.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.13.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.13.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.13.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.13.5.1.1.2.1.1.1">MSD <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib2" title="">antonelli2022medical </a></cite> &amp; KiTS <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib38" title="">heller2021state </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.13.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.13.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.13.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.13.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.13.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.13.6.1.1.1" style="font-size:70%;">Diffusion Model</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.14">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.14.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.14.1.1">
<span class="ltx_p" id="Ch0.T1.4.14.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.14.1.1.1.1" style="font-size:70%;">Lai et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib59" title="">lai2024pixel </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.14.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.14.2.1">
<span class="ltx_p" id="Ch0.T1.4.14.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.14.2.1.1.1" style="font-size:70%;">abdomen</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.14.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.14.3.1">
<span class="ltx_p" id="Ch0.T1.4.14.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.14.3.1.1.1" style="font-size:70%;">tumors in liver, pancreas and kidney</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.14.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.14.4.1">
<span class="ltx_p" id="Ch0.T1.4.14.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.14.4.1.1.1" style="font-size:70%;">CT</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.14.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.14.5.1">
<span class="ltx_p" id="Ch0.T1.4.14.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.14.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.14.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.14.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.14.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.14.5.1.1.2.1.1.1">MSD <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib2" title="">antonelli2022medical </a></cite> &amp; KiTS <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib38" title="">heller2021state </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.14.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.14.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.14.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.14.6.1">
<span class="ltx_p" id="Ch0.T1.4.14.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.14.6.1.1.1" style="font-size:70%;">hand-crafted designs</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.15" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.15.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.15.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.15.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.15.1.1.1.1" style="font-size:70%;">Yu et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib117" title="">yu20183d </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.15.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.15.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.15.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.15.2.1.1.1" style="font-size:70%;">brain</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.15.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.15.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.15.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.15.3.1.1.1" style="font-size:70%;">brain tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.15.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.15.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.15.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.15.4.1.1.1" style="font-size:70%;">MRI</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.15.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.15.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.15.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.15.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.15.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.15.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.15.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.15.5.1.1.2.1.1.1">BraTS <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib72" title="">menze2014multimodal </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.15.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.15.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.15.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.15.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.15.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.15.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.16">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.16.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.16.1.1">
<span class="ltx_p" id="Ch0.T1.4.16.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.16.1.1.1.1" style="font-size:70%;">Han et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib34" title="">han2019infinite </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.16.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.16.2.1">
<span class="ltx_p" id="Ch0.T1.4.16.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.16.2.1.1.1" style="font-size:70%;">brain</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.16.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.16.3.1">
<span class="ltx_p" id="Ch0.T1.4.16.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.16.3.1.1.1" style="font-size:70%;">brain tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.16.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.16.4.1">
<span class="ltx_p" id="Ch0.T1.4.16.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.16.4.1.1.1" style="font-size:70%;">MRI</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.16.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.16.5.1">
<span class="ltx_p" id="Ch0.T1.4.16.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.16.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.16.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.16.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.16.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.16.5.1.1.2.1.1.1">BraTS <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib72" title="">menze2014multimodal </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.16.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.16.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.16.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.16.6.1">
<span class="ltx_p" id="Ch0.T1.4.16.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.16.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.17" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.17.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.17.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.17.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.17.1.1.1.1" style="font-size:70%;">Zhao et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib119" title="">zhao2020tripartite </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.17.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.17.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.17.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.17.2.1.1.1" style="font-size:70%;">abdomen</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.17.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.17.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.17.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.17.3.1.1.1" style="font-size:70%;">liver tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.17.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.17.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.17.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.17.4.1.1.1" style="font-size:70%;">MRI</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.17.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.17.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.17.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.17.5.1.1.1" style="font-size:70%;">private dataset</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.17.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.17.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.17.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.17.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.18">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.18.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.18.1.1">
<span class="ltx_p" id="Ch0.T1.4.18.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.18.1.1.1.1" style="font-size:70%;">Mukherkjee et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib73" title="">mukherkjee2022brain </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.18.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.18.2.1">
<span class="ltx_p" id="Ch0.T1.4.18.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.18.2.1.1.1" style="font-size:70%;">brain</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.18.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.18.3.1">
<span class="ltx_p" id="Ch0.T1.4.18.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.18.3.1.1.1" style="font-size:70%;">brain tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.18.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.18.4.1">
<span class="ltx_p" id="Ch0.T1.4.18.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.18.4.1.1.1" style="font-size:70%;">MRI</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.18.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.18.5.1">
<span class="ltx_p" id="Ch0.T1.4.18.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.18.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.18.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.18.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.18.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.18.5.1.1.2.1.1.1">BraTS <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib72" title="">menze2014multimodal </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.18.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.18.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.18.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.18.6.1">
<span class="ltx_p" id="Ch0.T1.4.18.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.18.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.19" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.19.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.19.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.19.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.19.1.1.1.1" style="font-size:70%;">Basaran et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib6" title="">basaran2023lesionmix </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.19.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.19.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.19.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.19.2.1.1.1" style="font-size:70%;">brain</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.19.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.19.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.19.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.19.3.1.1.1" style="font-size:70%;">brain tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.19.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.19.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.19.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.19.4.1.1.1" style="font-size:70%;">MRI</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.19.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.19.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.19.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.19.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.19.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.19.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.19.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.19.5.1.1.2.1.1.1">WMH <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib54" title="">kuijf2019standardized </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.19.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.19.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.19.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.19.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.19.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.19.6.1.1.1" style="font-size:70%;">hand-crafted designs</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.20">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.20.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.20.1.1">
<span class="ltx_p" id="Ch0.T1.4.20.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.20.1.1.1.1" style="font-size:70%;">Wang et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib103" title="">wang2018locality </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.20.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.20.2.1">
<span class="ltx_p" id="Ch0.T1.4.20.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.20.2.1.1.1" style="font-size:70%;">brain</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.20.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.20.3.1">
<span class="ltx_p" id="Ch0.T1.4.20.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.20.3.1.1.1" style="font-size:70%;">-</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.20.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.20.4.1">
<span class="ltx_p" id="Ch0.T1.4.20.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.20.4.1.1.1" style="font-size:70%;">PET</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.20.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.20.5.1">
<span class="ltx_p" id="Ch0.T1.4.20.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.20.5.1.1.1" style="font-size:70%;">private dataser</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.20.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.20.6.1">
<span class="ltx_p" id="Ch0.T1.4.20.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.20.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.21" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.21.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.21.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.21.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.21.1.1.1.1" style="font-size:70%;">Luo et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib69" title="">luo2022adaptive </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.21.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.21.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.21.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.21.2.1.1.1" style="font-size:70%;">brain</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.21.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.21.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.21.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.21.3.1.1.1" style="font-size:70%;">-</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.21.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.21.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.21.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.21.4.1.1.1" style="font-size:70%;">PET</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.21.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.21.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.21.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.21.5.1.1.1" style="font-size:70%;">private dataset</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.21.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.21.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.21.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.21.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.22">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.22.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.22.1.1">
<span class="ltx_p" id="Ch0.T1.4.22.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.22.1.1.1.1" style="font-size:70%;">Sharan et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib88" title="">sharan2021mutually </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.22.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.22.2.1">
<span class="ltx_p" id="Ch0.T1.4.22.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.22.2.1.1.1" style="font-size:70%;">mitral valve</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.22.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.22.3.1">
<span class="ltx_p" id="Ch0.T1.4.22.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.22.3.1.1.1" style="font-size:70%;">landmark detection</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.22.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.22.4.1">
<span class="ltx_p" id="Ch0.T1.4.22.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.22.4.1.1.1" style="font-size:70%;">Endo.</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.22.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.22.5.1">
<span class="ltx_p" id="Ch0.T1.4.22.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.22.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.22.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.22.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.22.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.22.5.1.1.2.1.1.1">surgical simulator <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib23" title="">engelhardt2019flexible </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.22.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.22.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.22.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.22.6.1">
<span class="ltx_p" id="Ch0.T1.4.22.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.22.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.23" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.23.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.23.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.23.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.23.1.1.1.1" style="font-size:70%;">Yoon et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib116" title="">yoon2022colonoscopic </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.23.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.23.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.23.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.23.2.1.1.1" style="font-size:70%;">colon</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.23.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.23.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.23.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.23.3.1.1.1" style="font-size:70%;">polyp detection</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.23.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.23.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.23.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.23.4.1.1.1" style="font-size:70%;">Endo.</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.23.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.23.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.23.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.23.5.1.1.1" style="font-size:70%;">private dataset</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.23.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.23.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.23.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.23.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.24">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.24.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.24.1.1">
<span class="ltx_p" id="Ch0.T1.4.24.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.24.1.1.1.1" style="font-size:70%;">Hou et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib41" title="">hou2019robust </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.24.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.24.2.1">
<span class="ltx_p" id="Ch0.T1.4.24.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.24.2.1.1.1" style="font-size:70%;">tissue</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.24.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.24.3.1">
<span class="ltx_p" id="Ch0.T1.4.24.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.24.3.1.1.1" style="font-size:70%;">cancer</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.24.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.24.4.1">
<span class="ltx_p" id="Ch0.T1.4.24.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.24.4.1.1.1" style="font-size:70%;">Histo.</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.24.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.24.5.1">
<span class="ltx_p" id="Ch0.T1.4.24.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.24.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.24.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.24.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.24.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.24.5.1.1.2.1.1.1">Kumar <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib55" title="">kumar2017dataset </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.24.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.24.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.24.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.24.6.1">
<span class="ltx_p" id="Ch0.T1.4.24.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.24.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.25" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.25.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.25.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.25.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.25.1.1.1.1" style="font-size:70%;">Xue et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib110" title="">xue2021selective </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.25.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.25.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.25.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.25.2.1.1.1" style="font-size:70%;">tissue</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.25.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.25.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.25.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.25.3.1.1.1" style="font-size:70%;">cancer</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.25.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.25.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.25.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.25.4.1.1.1" style="font-size:70%;">Histo.</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.25.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.25.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.25.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.25.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.25.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.25.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.25.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.25.5.1.1.2.1.1.1">PCam <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib100" title="">veeling2018rotation </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.25.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.25.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.25.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.25.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.25.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.25.6.1.1.1" style="font-size:70%;">GAN</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.26">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.26.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.26.1.1">
<span class="ltx_p" id="Ch0.T1.4.26.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.26.1.1.1.1" style="font-size:70%;">Aversa et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib4" title="">aversa2024diffinfinite </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.26.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.26.2.1">
<span class="ltx_p" id="Ch0.T1.4.26.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.26.2.1.1.1" style="font-size:70%;">tissue</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.26.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.26.3.1">
<span class="ltx_p" id="Ch0.T1.4.26.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.26.3.1.1.1" style="font-size:70%;">cancer</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.26.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.26.4.1">
<span class="ltx_p" id="Ch0.T1.4.26.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.26.4.1.1.1" style="font-size:70%;">Histo.</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.26.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.26.5.1">
<span class="ltx_p" id="Ch0.T1.4.26.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.26.5.1.1.1" style="font-size:70%;">private dataset</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T1.4.26.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.26.6.1">
<span class="ltx_p" id="Ch0.T1.4.26.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.26.6.1.1.1" style="font-size:70%;">Diffusion Model</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T1.4.27" style="background-color:#F2F2F2;">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T1.4.27.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.27.1.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.27.1.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.27.1.1.1.1" style="font-size:70%;">Du et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib19" title="">du2023boosting </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T1.4.27.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.27.2.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.27.2.1.1" style="width:47.7pt;"><span class="ltx_text" id="Ch0.T1.4.27.2.1.1.1" style="font-size:70%;">skin</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T1.4.27.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.27.3.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.27.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.27.3.1.1.1" style="font-size:70%;">dermatoscopic lesion</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T1.4.27.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.27.4.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.27.4.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T1.4.27.4.1.1.1" style="font-size:70%;">Dermo.</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T1.4.27.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.27.5.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.27.5.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T1.4.27.5.1.1.1"></span><span class="ltx_text" id="Ch0.T1.4.27.5.1.1.2" style="font-size:70%;">
<span class="ltx_tabular ltx_align_top" id="Ch0.T1.4.27.5.1.1.2.1">
<span class="ltx_tr" id="Ch0.T1.4.27.5.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T1.4.27.5.1.1.2.1.1.1">ISIC <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib17" title="">codella2018skin </a></cite></span></span>
</span></span><span class="ltx_text" id="Ch0.T1.4.27.5.1.1.3"></span><span class="ltx_text" id="Ch0.T1.4.27.5.1.1.4" style="font-size:70%;"></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T1.4.27.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T1.4.27.6.1" style="background-color:#F2F2F2;">
<span class="ltx_p" id="Ch0.T1.4.27.6.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T1.4.27.6.1.1.1" style="font-size:70%;">Diffusion Model</span></span>
</span>
</td>
</tr>
</table>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<ul class="ltx_itemize ltx_figure_panel" id="Ch0.S1.I1">
<li class="ltx_item" id="Ch0.S1.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S1.I1.i1.p1">
<p class="ltx_p" id="Ch0.S1.I1.i1.p1.1"><span class="ltx_text" id="Ch0.S1.I1.i1.p1.1.1" style="font-size:70%;">GAN stands for Generative Adversarial Network</span></p>
</div>
</li>
<li class="ltx_item" id="Ch0.S1.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S1.I1.i2.p1">
<p class="ltx_p" id="Ch0.S1.I1.i2.p1.1"><span class="ltx_text" id="Ch0.S1.I1.i2.p1.1.1" style="font-size:70%;">Endo. refers to Endoscopy, Histo. refers to Histopathology, and Dermo. refers to dermoscopy.</span></p>
</div>
</li>
</ul>
</div>
</div>
</figure>
</section>
<section class="ltx_subsection" id="Ch0.S1.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">1.2 </span>Real vs. Synthetic Data</h3>
<div class="ltx_para" id="Ch0.S1.SS2.p1">
<p class="ltx_p" id="Ch0.S1.SS2.p1.1">Unlike real data, which are collected from real-world imaging devices and represent true observations, synthetic data are created using algorithms, simulations, or models designed to replicate the properties of real data. We summarize the differences between real data and synthetic data as follows:
<span class="ltx_text ltx_font_italic" id="Ch0.S1.SS2.p1.1.1">First</span>, real data are collected from actual imaging devices (e.g., X-ray, CT, MRI, PET, ultrasound, histopathology). Synthetic data are generated artificially using computational methods and simulations <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib42" title="">hu2023label </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib59" title="">lai2024pixel </a></cite>.
<span class="ltx_text ltx_font_italic" id="Ch0.S1.SS2.p1.1.2">Second</span>, real data are often limited by practical constraints such as patient privacy, the cost of imaging, and the time required for data collection and annotation <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib13" title="">chou2024acquiring </a></cite>. Synthetic data can be produced in large quantities without the ethical and logistical issues associated with real data collection <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib10" title="">chiruvella2021ethical </a></cite>.
<span class="ltx_text ltx_font_italic" id="Ch0.S1.SS2.p1.1.3">Third</span>, real data require manual annotation by experts, which is time-consuming <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib78" title="">park2020annotated </a></cite> and prone to human error <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib65" title="">li2024abdomenatlas </a></cite>. Annotations of synthetic data can be automatically generated as part of the data creation process, ensuring consistency and accuracy <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib80" title="">qu2024abdomenatlas </a></cite>.</p>
</div>
</section>
</section>
<section class="ltx_section" id="Ch0.S2">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Detecting Real Tumors in CT Scans</h2>
<figure class="ltx_figure" id="Ch0.F2"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_square" height="549" id="Ch0.F2.g1" src="extracted/5843742/fig_solid_organ_tumor.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>Tumors in solid organs. (a) Hepatocellular carcinoma. (b) Thymoma. (c) Solid pseudopapillary tumor of the pancreas. (d) Pancreatic mucinous cystadenoma. (e) Pancreatic mucinous cystadenocarcinoma. (f) Pancreatic adenocarcinoma. (g) Neuroendocrine tumor in liver. (h) Meningioma. (i) Mediastinal lymphoma.
</figcaption>
</figure>
<section class="ltx_subsection" id="Ch0.S2.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.1 </span>Tumors in Solid Organs</h3>
<div class="ltx_para" id="Ch0.S2.SS1.p1">
<p class="ltx_p" id="Ch0.S2.SS1.p1.1">Solid tumors in organs like the liver, kidneys, and brain—such as hepatocellular carcinoma, renal cell carcinoma, and glioma—typically show well-defined margins and growth patterns <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib85" title="">sahani2016abdominal </a></cite>, illustrated in Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F2" title="Figure 2 ‣ 2 Detecting Real Tumors in CT Scans ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">2</span></a>. In CT images, early-stage tumors appear as small nodules with slightly blurred edges and homogeneous texture. As tumors advance, they grow larger, become irregular in shape, and exhibit significant mass effect and infiltrative growth <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib20" title="">dunnick2016renal </a></cite>. Advanced tumors may also show hemorrhage, necrosis, and fibrosis, leading to a heterogeneous appearance <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib89" title="">silverman2012oncologic </a></cite>.</p>
</div>
<div class="ltx_para" id="Ch0.S2.SS1.p2">
<ul class="ltx_itemize" id="Ch0.S2.I1">
<li class="ltx_item" id="Ch0.S2.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S2.I1.i1.p1">
<p class="ltx_p" id="Ch0.S2.I1.i1.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S2.I1.i1.p1.1.1">Liver tumors:</span> Hepatocellular carcinoma (HCC) is the most common malignant liver tumor. Early HCC presents as a small, well-differentiated nodule with a good prognosis and low metastatic potential <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib25" title="">fowler2021pathologic </a></cite>. CT imaging may show mass effect extending beyond the liver, displacement of blood vessels, intrahepatic venous thrombosis, and bile duct obstruction <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib82" title="">reynolds2015infiltrative </a></cite>. HCCs often exhibit intense arterial-phase enhancement due to neoangiogenesis and reduced portal triads, and typically appear hypoattenuating on venous phase scans <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib5" title="">ayuso2018diagnosis </a></cite>.</p>
</div>
</li>
<li class="ltx_item" id="Ch0.S2.I1.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S2.I1.i2.p1">
<p class="ltx_p" id="Ch0.S2.I1.i2.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S2.I1.i2.p1.1.1">Pancreatic tumors:</span> Pancreatic ductal adenocarcinoma (PDAC) constitutes the majority of malignant pancreatic tumors and is associated with a very poor prognosis and high morbidity. In the early stages, PDACs typically appear as homogeneous small nodules with blurred edges. Secondary findings associated with advanced PDACs include contour abnormalities, abrupt termination of the biliary or pancreatic duct, pancreatic atrophy upstream from the mass, vascular encasement, etc. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib57" title="">laeseke2015combining </a></cite>. PDACs typically exhibit poor enhancement, appearing hypoattenuating relative to the surrounding pancreatic parenchyma. This hypoenhancement is attributed to the development of a dense fibroblastic stromal component in PDACs <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib22" title="">elbanna2020imaging </a></cite>.</p>
</div>
</li>
<li class="ltx_item" id="Ch0.S2.I1.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S2.I1.i3.p1">
<p class="ltx_p" id="Ch0.S2.I1.i3.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S2.I1.i3.p1.1.1">Kidney tumors:</span> Renal cell carcinoma (RCC) is the most common adult renal epithelial cancer, accounting for more than 90% of all renal malignancies <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib61" title="">leveridge2010imaging </a></cite>. The most prevalent subtype, clear cell RCC, presents as a homogeneously enhancing lesion during the corticomedullary phase and as a hypoattenuating renal lesion surrounded by homogenously enhancing renal parenchyma in the nephrographic phase. Advanced clear cell RCC often appears heterogeneous in imaging due to the presence of hemorrhage, necrosis, and cysts, along with invasion into the renal pelvis, perirenal fat, or renal vessels <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib104" title="">pmid29668296 </a></cite>.</p>
</div>
</li>
</ul>
</div>
</section>
<section class="ltx_subsection" id="Ch0.S2.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.2 </span>Tumors in Tubular Organs</h3>
<figure class="ltx_figure" id="Ch0.F3"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_square" height="535" id="Ch0.F3.g1" src="extracted/5843742/fig_tubular_organ_tumor.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 3: </span>Tumors in tubular organs. (a) Gastrointestinal stromal tumor. (b) Sigmoid colon cancer. (c) Gastric cancer. (d) Lung metastases. (e) Intestinal carcinoid tumor. (f) Gallbladder carcinoma. (g) Gallbladder adenocarcinoma. (h) Colon cancer. (i) Cholangiocarcinoma.</figcaption>
</figure>
<div class="ltx_para" id="Ch0.S2.SS2.p1">
<p class="ltx_p" id="Ch0.S2.SS2.p1.1">Tumors in tubular organs, illustrated in Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F3" title="Figure 3 ‣ 2.2 Tumors in Tubular Organs ‣ 2 Detecting Real Tumors in CT Scans ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">3</span></a>, have distinct growth patterns <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib74" title="">nerad2016diagnostic </a></cite>: exophytic, where the tumor expands into the lumen, and invasive, where it penetrates the organ wall into adjacent structures. For instance, colon cancer progresses from stage 0, confined to the lining, to stage I, invading the submucosa, and stage II, extending through the wall without nearby invasion <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib21" title="">edge2010american </a></cite>. Stage III involves spread to lymph nodes, and stage IV features metastasis to distant organs such as the liver or lungs. We also highlight unique characteristics of representative tubular tumors to illustrate their specific behaviors and progression.</p>
</div>
<div class="ltx_para" id="Ch0.S2.SS2.p2">
<ul class="ltx_itemize" id="Ch0.S2.I2">
<li class="ltx_item" id="Ch0.S2.I2.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S2.I2.i1.p1">
<p class="ltx_p" id="Ch0.S2.I2.i1.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S2.I2.i1.p1.1.1">Esophageal tumors</span>, though rare, have a poor prognosis if malignant unless detected early and surgically removed <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib45" title="">iyer2004imaging </a></cite>. Imaging studies include X-ray esophagography, CT, endoscopic ultrasound, and PET. Malignant strictures often show asymmetric narrowing with abrupt margins and irregular, nodular, or ulcerated surfaces. X-ray esophagography helps evaluate invasion of the muscularis mucosae for early-stage cancers. Key CT features are eccentric or circumferential wall thickening over 5 mm and periesophageal soft tissue and fat stranding <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib62" title="">lewin1996tumors </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib94" title="">stout1957tumors </a></cite>.</p>
</div>
</li>
<li class="ltx_item" id="Ch0.S2.I2.i2" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S2.I2.i2.p1">
<p class="ltx_p" id="Ch0.S2.I2.i2.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S2.I2.i2.p1.1.1">Stomach tumors</span>, primarily adenocarcinoma, are common and often asymptomatic when superficial. Up to 50% of patients may have nonspecific gastrointestinal symptoms like dyspepsia. Endoscopy is the most sensitive method for diagnosis, allowing direct visualization and biopsy. Initial detection is often through radiological methods, with CT imaging using negative contrast to reveal common features such as polypoid masses, wall thickening, or ulceration <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib93" title="">stout1953tumors </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib18" title="">davis2000tumors </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib62" title="">lewin1996tumors </a></cite>.</p>
</div>
</li>
<li class="ltx_item" id="Ch0.S2.I2.i3" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S2.I2.i3.p1">
<p class="ltx_p" id="Ch0.S2.I2.i3.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S2.I2.i3.p1.1.1">Colorectal tumors</span> are a leading gastrointestinal malignancy. Contrast-enhanced CT of the chest, abdomen, and pelvis is used for staging, detecting metastases, evaluating surgical options, and assessing treatment response. Colorectal cancers typically appear as soft tissue masses that narrow the bowel lumen. Larger tumors often show ulceration, mucinous tumors may appear as low-density masses with low-density lymph nodes, and psammomatous calcifications can be seen in mucinous adenocarcinoma. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib92" title="">soga2005early </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib30" title="">griswold1975colon </a></cite>.</p>
</div>
</li>
</ul>
</div>
<figure class="ltx_figure" id="Ch0.F4"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="222" id="Ch0.F4.g1" src="extracted/5843742/fig_preliminary.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 4: </span>
<span class="ltx_text ltx_font_italic" id="Ch0.F4.2.1">Feature analysis and reader study.</span> The left panel features a t-SNE (t-distributed stochastic neighbor embedding) visualization that maps the multidimensional Radiomics features of tumors from the liver, pancreas, and kidneys onto a two-dimensional space. This visualization underscores the substantial overlap in features among early-stage tumors from different organs, which may contribute to the challenges in correctly identifying their organ types. Complementing these findings, this study evaluates the efficacy of a support vector machine (SVM) classifier, which utilizes Radiomics Features <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib16" title="">chu2019utility </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib102" title="">wang2017comparison </a></cite>, in differentiating the organ types for the cropped tumors. The SVM classifier is trained to classify each tumor as originating from either the liver, pancreas, or kidneys—a three-way classification challenge. Parallel to the assessment of the SVM classifier, three expert radiologists conducted a similar evaluation by reviewing the original CT scans containing these tumors. The results displayed on the right panel reveal significant difficulties faced by both the SVM classifier and the radiologists when it comes to accurately pinpointing the origin of early-stage tumors. The precision and recall metrics for both the machine and human methods approximate the performance expected from random selection.
</figcaption>
</figure>
</section>
<section class="ltx_subsection" id="Ch0.S2.SS3">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">2.3 </span>High Similarity in Early-Stage Tumors</h3>
<div class="ltx_para" id="Ch0.S2.SS3.p1">
<p class="ltx_p" id="Ch0.S2.SS3.p1.1">Early-stage tumors (<math alttext="&lt;" class="ltx_Math" display="inline" id="Ch0.S2.SS3.p1.1.m1.1"><semantics id="Ch0.S2.SS3.p1.1.m1.1a"><mo id="Ch0.S2.SS3.p1.1.m1.1.1" xref="Ch0.S2.SS3.p1.1.m1.1.1.cmml">&lt;</mo><annotation-xml encoding="MathML-Content" id="Ch0.S2.SS3.p1.1.m1.1b"><lt id="Ch0.S2.SS3.p1.1.m1.1.1.cmml" xref="Ch0.S2.SS3.p1.1.m1.1.1"></lt></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S2.SS3.p1.1.m1.1c">&lt;</annotation><annotation encoding="application/x-llamapun" id="Ch0.S2.SS3.p1.1.m1.1d">&lt;</annotation></semantics></math> 2cm) frequently exhibit similar imaging characteristics in CT scans, regardless of whether they originate in the liver, pancreas, or kidneys <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib12" title="">choi2014ct </a></cite>. Should this finding be validated, it could carry profound implications for the application of generative AI in medical imaging. This implies that both modeling-based and learning-based approaches could be developed on a single tumor type with readily available annotated data and subsequently applied to synthesize various tumor types in other organs, for which data/annotation acquisition is more arduous.</p>
</div>
<div class="ltx_para" id="Ch0.S2.SS3.p2">
<p class="ltx_p" id="Ch0.S2.SS3.p2.1">A study involving three expert radiologists was conducted to assess their ability to recognize the organ class of early-stage cancers <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib9" title="">chen2024towards </a></cite>. Three expert radiologists, certified in accordance with the Quality Standards Act, participated in the reader study. Recognition results are presented in Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F4" title="Figure 4 ‣ 2.2 Tumors in Tubular Organs ‣ 2 Detecting Real Tumors in CT Scans ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">4</span></a>(b). The results were sufficiently compelling that medical professionals, boasting over five years of experience, could potentially mistake the synthetic tumors for genuine ones. The precision and recall scores, which approximate randomness, imply that the similarity in the appearance of early-stage tumors is such that even seasoned radiologists encounter difficulties when attempting to distinguish the organ types of these tumors.</p>
</div>
<div class="ltx_para" id="Ch0.S2.SS3.p3">
<p class="ltx_p" id="Ch0.S2.SS3.p3.1">The similarity of early-stage tumors can be evidenced by their Radiomics Feature<span class="ltx_note ltx_role_footnote" id="Ch0.footnote1"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup><span class="ltx_tag ltx_tag_note">1</span>Utilization of the official radiomics feature repository <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib99" title="">van2017computational </a></cite> enables the extraction of appearance features, comprising 3D shape-based features, gray level co-occurrence matrix, gray level run length matrix, gray level size zone matrix, neighboring gray-tone difference matrix, and gray level dependence matrix.</span></span></span> profiles <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib9" title="">chen2024towards </a></cite>. From a <span class="ltx_text ltx_font_italic" id="Ch0.S2.SS3.p3.1.1">qualitative</span> standpoint, Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F4" title="Figure 4 ‣ 2.2 Tumors in Tubular Organs ‣ 2 Detecting Real Tumors in CT Scans ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">4</span></a>(a) depicts the feature mapping in a two-dimensional space via <span class="ltx_text ltx_font_italic" id="Ch0.S2.SS3.p3.1.2">t</span>-SNE. The appearance features of early-stage tumors manifest within a joint feature space, with no discernible segregation among different organ types. From a <span class="ltx_text ltx_font_italic" id="Ch0.S2.SS3.p3.1.3">quantitative</span> perspective, we trained a support vector machine (SVM) classifier to identify the organ types of early-stage tumors. To infer a general conclusion, we conducted ten repeated experiments and computed the precision and recall metrics of the SVM classifier for both the training and test sets. The final results indicate that both the precision and recall metrics for the training set are close to 1.0, demonstrating that the SVM is effectively trained and has established a robust decision boundary for the training set. However, the precision scores for the test set approximate random chance, as illustrated in Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F4" title="Figure 4 ‣ 2.2 Tumors in Tubular Organs ‣ 2 Detecting Real Tumors in CT Scans ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">4</span></a>(b). This implies that even an effectively trained SVM classifier encounters difficulty in recognizing the organ types of unseen early-stage tumors.</p>
</div>
</section>
</section>
<section class="ltx_section" id="Ch0.S3">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Technical Barriers and Clinical Needs</h2>
<section class="ltx_subsection" id="Ch0.S3.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.1 </span>Technical Barriers</h3>
<div class="ltx_para" id="Ch0.S3.SS1.p1">
<p class="ltx_p" id="Ch0.S3.SS1.p1.1">AI development for real tumors faces key technical barriers:
First, <span class="ltx_text ltx_font_bold" id="Ch0.S3.SS1.p1.1.1">data scarcity</span>: High-performance models need extensive annotated data, which is limited due to the time and expertise required for medical image and genomic annotation. Rare cancers further exacerbate this issue, leading to poor model performance on less common types <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib120" title="">zhou2021towards </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib121" title="">zhou2022interpreting </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib96" title="">tang2024efficient </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib14" title="">chou2024embracing </a></cite>.
Second, <span class="ltx_text ltx_font_bold" id="Ch0.S3.SS1.p1.1.2">generalization to different organs</span>: AI models struggle to generalize across organs due to distinct anatomical structures and imaging modalities. Models trained on one organ, like the lung, perform poorly on others, such as the liver, due to differing tissue compositions and imaging techniques <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib118" title="">zhang2023continual </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib68" title="">liu2024universal </a></cite>.
Third, <span class="ltx_text ltx_font_bold" id="Ch0.S3.SS1.p1.1.3">generalization to different demographics</span>: Privacy laws restrict access to diverse datasets, impacting model robustness. Variations in imaging protocols and genetic differences across populations can lead to biased models that perform poorly on underrepresented groups <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib66" title="">li2024well </a></cite>.</p>
</div>
</section>
<section class="ltx_subsection" id="Ch0.S3.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">3.2 </span>Clinical Needs</h3>
<div class="ltx_para" id="Ch0.S3.SS2.p1">
<p class="ltx_p" id="Ch0.S3.SS2.p1.1">Cancer research addresses critical clinical needs to improve patient outcomes and advance oncology. Key objectives include early cancer detection, developing effective treatments, and personalizing care strategies to enhance treatment success and system efficiency.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S3.SS2.p2">
<p class="ltx_p" id="Ch0.S3.SS2.p2.1"><span class="ltx_text ltx_font_bold" id="Ch0.S3.SS2.p2.1.1">Early Detection and Diagnosis:</span>
Early detection and diagnosis of cancer are crucial for improving patient outcomes, as identifying cancer at an early stage often leads to more effective treatment and better survival rates. There is a critical need for screening methods with high sensitivity (ability to correctly identify those with cancer) and high specificity (ability to correctly identify those without cancer). Improved accuracy reduces false positives and false negatives, which are common issues in current screening practices. Besides, enhanced accuracy can help avoid overdiagnosis, where non-life-threatening cancers are treated unnecessarily, causing undue stress and potential harm to patients.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S3.SS2.p3">
<p class="ltx_p" id="Ch0.S3.SS2.p3.9"><span class="ltx_text ltx_font_bold" id="Ch0.S3.SS2.p3.9.1">Health System Efficiency:</span> For effective training in tumor detection across multiple organs, AI models traditionally require numerous annotated real tumor examples from each organ <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib123" title="">zhu2022assembling </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib50" title="">kang2023label </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib79" title="">qu2023annotating </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib67" title="">liu2023clip </a></cite>. However, these AI models often face difficulties in generalizing their ability to interpret images from different hospitals, a challenge compounded by varying imaging protocols, patient demographics, and scanner manufacturers <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib75" title="">orbes2019multi </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib111" title="">yan2020mri </a></cite>. While the challenge of domain generalization could be alleviated if the AI is trained on a considerable number of annotated data from various domains <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib109" title="">xiao2022catenorm </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib114" title="">yao2022unsupervised </a></cite>, it could take up to 25 human years for just annotating tumors in a specific organ <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib108" title="">xia2022felix </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib1" title="">abi2023automatic </a></cite>. Collecting and annotating a comprehensive dataset that includes tumor examples from several organs (<math alttext="N" class="ltx_Math" display="inline" id="Ch0.S3.SS2.p3.1.m1.1"><semantics id="Ch0.S3.SS2.p3.1.m1.1a"><mi id="Ch0.S3.SS2.p3.1.m1.1.1" xref="Ch0.S3.SS2.p3.1.m1.1.1.cmml">N</mi><annotation-xml encoding="MathML-Content" id="Ch0.S3.SS2.p3.1.m1.1b"><ci id="Ch0.S3.SS2.p3.1.m1.1.1.cmml" xref="Ch0.S3.SS2.p3.1.m1.1.1">𝑁</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S3.SS2.p3.1.m1.1c">N</annotation><annotation encoding="application/x-llamapun" id="Ch0.S3.SS2.p3.1.m1.1d">italic_N</annotation></semantics></math>) and numerous hospitals (<math alttext="M" class="ltx_Math" display="inline" id="Ch0.S3.SS2.p3.2.m2.1"><semantics id="Ch0.S3.SS2.p3.2.m2.1a"><mi id="Ch0.S3.SS2.p3.2.m2.1.1" xref="Ch0.S3.SS2.p3.2.m2.1.1.cmml">M</mi><annotation-xml encoding="MathML-Content" id="Ch0.S3.SS2.p3.2.m2.1b"><ci id="Ch0.S3.SS2.p3.2.m2.1.1.cmml" xref="Ch0.S3.SS2.p3.2.m2.1.1">𝑀</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S3.SS2.p3.2.m2.1c">M</annotation><annotation encoding="application/x-llamapun" id="Ch0.S3.SS2.p3.2.m2.1d">italic_M</annotation></semantics></math>) is a formidable task, denoted by the complexity (<math alttext="N\times M" class="ltx_Math" display="inline" id="Ch0.S3.SS2.p3.3.m3.1"><semantics id="Ch0.S3.SS2.p3.3.m3.1a"><mrow id="Ch0.S3.SS2.p3.3.m3.1.1" xref="Ch0.S3.SS2.p3.3.m3.1.1.cmml"><mi id="Ch0.S3.SS2.p3.3.m3.1.1.2" xref="Ch0.S3.SS2.p3.3.m3.1.1.2.cmml">N</mi><mo id="Ch0.S3.SS2.p3.3.m3.1.1.1" lspace="0.222em" rspace="0.222em" xref="Ch0.S3.SS2.p3.3.m3.1.1.1.cmml">×</mo><mi id="Ch0.S3.SS2.p3.3.m3.1.1.3" xref="Ch0.S3.SS2.p3.3.m3.1.1.3.cmml">M</mi></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S3.SS2.p3.3.m3.1b"><apply id="Ch0.S3.SS2.p3.3.m3.1.1.cmml" xref="Ch0.S3.SS2.p3.3.m3.1.1"><times id="Ch0.S3.SS2.p3.3.m3.1.1.1.cmml" xref="Ch0.S3.SS2.p3.3.m3.1.1.1"></times><ci id="Ch0.S3.SS2.p3.3.m3.1.1.2.cmml" xref="Ch0.S3.SS2.p3.3.m3.1.1.2">𝑁</ci><ci id="Ch0.S3.SS2.p3.3.m3.1.1.3.cmml" xref="Ch0.S3.SS2.p3.3.m3.1.1.3">𝑀</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S3.SS2.p3.3.m3.1c">N\times M</annotation><annotation encoding="application/x-llamapun" id="Ch0.S3.SS2.p3.3.m3.1d">italic_N × italic_M</annotation></semantics></math>). We hypothesize that <span class="ltx_text ltx_font_italic" id="Ch0.S3.SS2.p3.9.2">tumor synthesis could address this challenge by creating various tumor types across non-tumor images from multiple hospitals, even if only one type of tumor is available and annotated.</span> This approach can simplify the complexity from <math alttext="N" class="ltx_Math" display="inline" id="Ch0.S3.SS2.p3.4.m4.1"><semantics id="Ch0.S3.SS2.p3.4.m4.1a"><mi id="Ch0.S3.SS2.p3.4.m4.1.1" xref="Ch0.S3.SS2.p3.4.m4.1.1.cmml">N</mi><annotation-xml encoding="MathML-Content" id="Ch0.S3.SS2.p3.4.m4.1b"><ci id="Ch0.S3.SS2.p3.4.m4.1.1.cmml" xref="Ch0.S3.SS2.p3.4.m4.1.1">𝑁</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S3.SS2.p3.4.m4.1c">N</annotation><annotation encoding="application/x-llamapun" id="Ch0.S3.SS2.p3.4.m4.1d">italic_N</annotation></semantics></math> <math alttext="\times" class="ltx_Math" display="inline" id="Ch0.S3.SS2.p3.5.m5.1"><semantics id="Ch0.S3.SS2.p3.5.m5.1a"><mo id="Ch0.S3.SS2.p3.5.m5.1.1" xref="Ch0.S3.SS2.p3.5.m5.1.1.cmml">×</mo><annotation-xml encoding="MathML-Content" id="Ch0.S3.SS2.p3.5.m5.1b"><times id="Ch0.S3.SS2.p3.5.m5.1.1.cmml" xref="Ch0.S3.SS2.p3.5.m5.1.1"></times></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S3.SS2.p3.5.m5.1c">\times</annotation><annotation encoding="application/x-llamapun" id="Ch0.S3.SS2.p3.5.m5.1d">×</annotation></semantics></math> <math alttext="M" class="ltx_Math" display="inline" id="Ch0.S3.SS2.p3.6.m6.1"><semantics id="Ch0.S3.SS2.p3.6.m6.1a"><mi id="Ch0.S3.SS2.p3.6.m6.1.1" xref="Ch0.S3.SS2.p3.6.m6.1.1.cmml">M</mi><annotation-xml encoding="MathML-Content" id="Ch0.S3.SS2.p3.6.m6.1b"><ci id="Ch0.S3.SS2.p3.6.m6.1.1.cmml" xref="Ch0.S3.SS2.p3.6.m6.1.1">𝑀</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S3.SS2.p3.6.m6.1c">M</annotation><annotation encoding="application/x-llamapun" id="Ch0.S3.SS2.p3.6.m6.1d">italic_M</annotation></semantics></math> to <math alttext="1" class="ltx_Math" display="inline" id="Ch0.S3.SS2.p3.7.m7.1"><semantics id="Ch0.S3.SS2.p3.7.m7.1a"><mn id="Ch0.S3.SS2.p3.7.m7.1.1" xref="Ch0.S3.SS2.p3.7.m7.1.1.cmml">1</mn><annotation-xml encoding="MathML-Content" id="Ch0.S3.SS2.p3.7.m7.1b"><cn id="Ch0.S3.SS2.p3.7.m7.1.1.cmml" type="integer" xref="Ch0.S3.SS2.p3.7.m7.1.1">1</cn></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S3.SS2.p3.7.m7.1c">1</annotation><annotation encoding="application/x-llamapun" id="Ch0.S3.SS2.p3.7.m7.1d">1</annotation></semantics></math> <math alttext="\times" class="ltx_Math" display="inline" id="Ch0.S3.SS2.p3.8.m8.1"><semantics id="Ch0.S3.SS2.p3.8.m8.1a"><mo id="Ch0.S3.SS2.p3.8.m8.1.1" xref="Ch0.S3.SS2.p3.8.m8.1.1.cmml">×</mo><annotation-xml encoding="MathML-Content" id="Ch0.S3.SS2.p3.8.m8.1b"><times id="Ch0.S3.SS2.p3.8.m8.1.1.cmml" xref="Ch0.S3.SS2.p3.8.m8.1.1"></times></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S3.SS2.p3.8.m8.1c">\times</annotation><annotation encoding="application/x-llamapun" id="Ch0.S3.SS2.p3.8.m8.1d">×</annotation></semantics></math> <math alttext="M" class="ltx_Math" display="inline" id="Ch0.S3.SS2.p3.9.m9.1"><semantics id="Ch0.S3.SS2.p3.9.m9.1a"><mi id="Ch0.S3.SS2.p3.9.m9.1.1" xref="Ch0.S3.SS2.p3.9.m9.1.1.cmml">M</mi><annotation-xml encoding="MathML-Content" id="Ch0.S3.SS2.p3.9.m9.1b"><ci id="Ch0.S3.SS2.p3.9.m9.1.1.cmml" xref="Ch0.S3.SS2.p3.9.m9.1.1">𝑀</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S3.SS2.p3.9.m9.1c">M</annotation><annotation encoding="application/x-llamapun" id="Ch0.S3.SS2.p3.9.m9.1d">italic_M</annotation></semantics></math>.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S3.SS2.p4">
<p class="ltx_p" id="Ch0.S3.SS2.p4.1"><span class="ltx_text ltx_font_bold" id="Ch0.S3.SS2.p4.1.1">Personalized Treatment Planning</span> Tumors within the same type of cancer can vary significantly at the genetic and molecular levels. Personalized treatment planning requires comprehensive genomic profiling to identify specific mutations, gene expression patterns, and other molecular characteristics that drive an individual’s cancer. This information helps in selecting targeted therapies that are more likely to be effective for that particular tumor profile. Understanding the diversity of cancer cells within a tumor can inform treatment strategies that target multiple pathways and cell populations simultaneously.</p>
</div>
</section>
</section>
<section class="ltx_section" id="Ch0.S4">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Technology Trend I: Modeling-based Approaches</h2>
<figure class="ltx_figure" id="Ch0.F5"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="162" id="Ch0.F5.g1" src="extracted/5843742/fig_CA_tumor.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 5: </span>Synthetic liver, pancreatic, and kidney tumors generated by Cellular Automata <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib59" title="">lai2024pixel </a></cite>.</figcaption>
</figure>
<div class="ltx_para" id="Ch0.S4.p1">
<p class="ltx_p" id="Ch0.S4.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S4.p1.1.1">Hand-crafted tumor synthesis</span> has been conducted in  <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib43" title="">hu2022synthetic </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib42" title="">hu2023label </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib44" title="">hu2023synthetic </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib63" title="">li2023early </a></cite>. This approach applies a sequence of hand-crafted morphological image-processing operations, including local selection, texture generation, shape generation, and post-processing, to generate realistic tumors for training AI models. The intrinsic observation about these operations is clinical knowledge. Taking liver tumors as an example, the mean HU intensity of hepatocellular carcinomas (tumors grown from liver cells) was 106 HU (with a range of 36-162 HU) <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib60" title="">lee2004triple </a></cite>. Milder carcinomas usually lead to smaller, fewer spherical lesions, while multi-focal lesions (which means scattered small tumors) only appear in rare cases. Additionally, larger tumors usually display evident mass effects and are accompanied by capsule appearances that separate the tumor from the liver parenchyma <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib71" title="">m2021use </a></cite>. This medical guidance, together with visual clues, determines the parameters and pipeline of this method.</p>
</div>
<figure class="ltx_figure" id="Ch0.F6"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="417" id="Ch0.F6.g1" src="extracted/5843742/fig_cellular.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 6: </span>
① Tumor development: The cellular automata simulate tumor growth from a single pixel to various sizes and even tumor death, producing synthetic tumors with diverse sizes, shapes, and textures. An idealized tumor is created to quantify development, with dead cells in the gray region, living quiescent cells in the inactive region, and proliferative cells in the active outer shell. ② Tissue quantification: The organ map in blue transforms CT images into distinct intensity levels affecting tumor development rate, while the red tumor map assigns values representing the tumor cell population. ③ Tumor interaction with boundaries and vessels: The tumor grows and exerts pressure against organ boundaries and deforms as it interacts with vessels. ④ Mapping synthetic tumors to CT images: A mapping function correlates the synthetic tumor with CT values, integrating the tumor’s state with the original CT intensity. ⑤ Training segmentation models with synthetic data: Pixel2Cancer generates new synthetic data for each epoch to train the segmentation model.
</figcaption>
</figure>
<div class="ltx_para" id="Ch0.S4.p2">
<p class="ltx_p" id="Ch0.S4.p2.1"><span class="ltx_text ltx_font_bold" id="Ch0.S4.p2.1.1">Cellular Automata</span> are computational models used to simulate complex systems through simple rules and interactions. They employ a grid of cells (pixels), where each cell is initially assigned a state between zero and ten to represent the tumor population. The basic element is the <span class="ltx_text ltx_font_italic" id="Ch0.S4.p2.1.2">cell</span>, which refers to a single <span class="ltx_text ltx_font_italic" id="Ch0.S4.p2.1.3">pixel</span> in the computed tomography (CT) image. Tumor growth and behavior are modeled based on specific rules that simulate processes such as proliferation, invasion, and death. These rules are derived from medical knowledge and are guided by an idealized tumor model that reflects real-world characteristics. The tumor state can then be integrated into the original CT images to generate synthetic tumors in different organs. This tumor synthesis approach allows for sampling tumors at various stages and analyzing tumor-organ interactions. Motivated by this, Lai et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib59" title="">lai2024pixel </a></cite> proposed Pixel2Cancer to simulate tumor growth (Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F6" title="Figure 6 ‣ 4 Technology Trend I: Modeling-based Approaches ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">6</span></a>). Tumors generated by Pixel2Cancer are illustrated in Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F5" title="Figure 5 ‣ 4 Technology Trend I: Modeling-based Approaches ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">5</span></a>.</p>
</div>
<section class="ltx_subsection" id="Ch0.S4.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.1 </span>Property of Pixel2Cancer</h3>
<div class="ltx_para ltx_noindent" id="Ch0.S4.SS1.p1">
<p class="ltx_p" id="Ch0.S4.SS1.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S4.SS1.p1.1.1">(i) Label-free.</span> Pixel2Cancer can be applied as a label-free data synthesis approach, eliminating the need for manual per-voxel annotation. Previous learning-based approaches, such as GANs <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib28" title="">goodfellow2014generative </a></cite> and Diffusion models <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib40" title="">ho2020denoising </a></cite>, are designed to learn the representation and distribution of tumors. While these approaches excel in generating natural images, synthesizing tumors in CT scans still requires significant amounts of paired tumor data. Moreover, when generating synthetic tumors, generative models also need masks to indicate the tumor locations and shapes <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib49" title="">jin2021free </a></cite>, necessitating extensive manual efforts for training and synthesis.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S4.SS1.p2">
<p class="ltx_p" id="Ch0.S4.SS1.p2.1"><span class="ltx_text ltx_font_bold" id="Ch0.S4.SS1.p2.1.1">(ii) Tumor development.</span> Pixel2Cancer incorporates specific medical knowledge regarding tumor growth and appearance, enabling the simulation of realistic tumors. None of the existing synthetic approaches can adequately simulate tumor development in abdominal CT, and the primary challenges in current synthetic methods are the proliferation and invasion of tumors <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib36" title="">harpold2007evolution </a></cite>. These processes in tumor growth are complex and interconnected, highly influenced by the surrounding environment <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib95" title="">tanase2015complexity </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib106" title="">wong2015pancreatic </a></cite>. Consequently, synthetic tumors generated using current methods may conflict with normal organ structures and pose challenges when adapting them to different organs.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S4.SS1.p3">
<p class="ltx_p" id="Ch0.S4.SS1.p3.1"><span class="ltx_text ltx_font_bold" id="Ch0.S4.SS1.p3.1.1">(iii) Early tumor detection &amp; boundary segmentation.</span>
Early detection of small tumors is critical for timely cancer diagnosis. However, real datasets often lack sufficient instances due to the asymptomatic nature of early-stage patients. Pixel2Cancer can generate more small tumors to improve the sensitivity of segmentation models for small tumor detection. Additionally, Pixel2Cancer generates synthetic tumors with precise tumor masks, whereas real data annotations are often inaccurate at the boundaries, leading to <span class="ltx_text ltx_font_italic" id="Ch0.S4.SS1.p3.1.2">label noise</span> in boundary segmentation accuracy.</p>
</div>
</section>
<section class="ltx_subsection" id="Ch0.S4.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">4.2 </span>Clinical Perspectives</h3>
<div class="ltx_para" id="Ch0.S4.SS2.p1">
<p class="ltx_p" id="Ch0.S4.SS2.p1.1">Tumors and genetic disorders from DNA mutations in single cells undergo complex growth processes <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib56" title="">kumar2017robbins </a></cite>. Mutations during cell division lead to uncontrolled proliferation, forming neoplastic lesions that can be benign or malignant <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib27" title="">golias2004cell </a></cite>. While both types follow similar growth principles, they differ in growth rate and invasiveness. Malignant tumors often grow rapidly, secreting growth factors or inducing surrounding stromal cells to do so, as seen in pancreatic IPMN lesions which grow larger and faster than benign ones <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib51" title="">kang2011cyst </a></cite>. Slow growth rates in renal tumors and hepatocellular carcinoma correlate with lower malignancy <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib91" title="">smaldone2012small </a></cite>. Malignant tumors are invasive, gradually penetrating and destroying surrounding tissues, whereas benign tumors remain confined to their original sites. Even slowly growing malignant tumors can infiltrate neighboring structures. Tumor necrosis, caused by rapid proliferation exceeding vascular supply <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib39" title="">hiraoka2010tumour </a></cite>, appears as irregular hypo-attenuating areas in CT images and serves as a poor prognostic indicator <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib25" title="">fowler2021pathologic </a></cite>. The <span class="ltx_text ltx_font_bold" id="Ch0.S4.SS2.p1.1.1">death</span> rule models this necrosis. A hybrid cellular automaton model is proposed to simulate tumor development from single cells to invasive tumors, capturing their continuous progression and interactions within the microenvironment.</p>
</div>
</section>
</section>
<section class="ltx_section" id="Ch0.S5">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Technology Trend II: Learning-based Approaches</h2>
<figure class="ltx_figure" id="Ch0.F7"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="249" id="Ch0.F7.g1" src="extracted/5843742/fig_DiffTumor_tumor.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 7: </span>Examples of synthetic tumors generated by DiffTumor <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib9" title="">chen2024towards </a></cite> on the liver, pancreas, and kidney.</figcaption>
</figure>
<div class="ltx_para" id="Ch0.S5.p1">
<p class="ltx_p" id="Ch0.S5.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S5.p1.1.1">Generative Adversarial Networks</span> (GANs) <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib29" title="">goodfellow2020generative </a></cite> have been extensively explored for generating synthetic tumors. Zhao et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib119" title="">zhao2020tripartite </a></cite> introduced Tripartite-GAN, which simultaneously achieves contrast-enhanced magnetic resonance imaging synthesis and tumor detection. Mukherkjee et al. <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib73" title="">mukherkjee2022brain </a></cite> proposed AGGrGAN to generate synthetic MRI scans of brain tumors. However, the training process of GANs is often unstable, making it challenging to achieve convergence. Additionally, GANs can suffer from issues such as mode collapse, where the model generates limited diversity in outputs.</p>
</div>
<div class="ltx_para" id="Ch0.S5.p2">
<p class="ltx_p" id="Ch0.S5.p2.1"><span class="ltx_text ltx_font_bold" id="Ch0.S5.p2.1.1">Diffusion Models</span> <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib76" title="">ozbey2023unsupervised </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib52" title="">khader2023denoising </a></cite> provide more stable and reliable training compared to GANs, as they gradually denoise data, making the optimization process easier to control. Additionally, Diffusion Models are capable of generating high-quality, diverse samples with fewer issues related to mode collapse. DiffTumor <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib9" title="">chen2024towards </a></cite> is the first to explore tumor synthesis in abdominal organs using Diffusion Models and demonstrates an efficient method for achieving generalizable tumor synthesis (Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F8" title="Figure 8 ‣ 5 Technology Trend II: Learning-based Approaches ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">8</span></a>). Tumors generated by DiffTumor are illustrated in Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F7" title="Figure 7 ‣ 5 Technology Trend II: Learning-based Approaches ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">7</span></a>.</p>
</div>
<figure class="ltx_figure" id="Ch0.F8"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="186" id="Ch0.F8.g1" src="extracted/5843742/fig_method_difftumor.png" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 8: </span>
<span class="ltx_text ltx_font_italic" id="Ch0.F8.2.1">Overview of DiffTumor framework.</span>
In pursuit of achieving generalizable tumor synthesis, DiffTumor encompasses three stages. ① The first stage is the training of an Autoencoder Model—comprising an encoder and a decoder—to learn comprehensive latent features. The learning objective in this stage entails image reconstruction conducted on 9,262 unlabeled three-dimensional CT volumes. Both the trained encoder and decoder are integral to subsequent stages. ② The second stage involves training a Diffusion Model—a specialized generative model—by utilizing latent features and tumor masks as conditions. Once trained, the model is capable of generating the requisite latent features for the reconstruction of CT volumes with tumors, utilizing arbitrary masks. ③ The third stage entails training a Segmentation Model with CT volumes of synthetic tumors, reconstructed by the decoder. Armed with a considerable repository of healthy CT volumes, DiffTumor has the capacity to generate an extensive collection of synthetic tumors, which vary in location, size, shape, texture, and intensity, thus contributing to the enhancement of AI models for tumor detection and segmentation.
</figcaption>
</figure>
<section class="ltx_subsection" id="Ch0.S5.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.1 </span>Property of DiffTumor</h3>
<div class="ltx_para ltx_noindent" id="Ch0.S5.SS1.p1">
<p class="ltx_p" id="Ch0.S5.SS1.p1.1"><span class="ltx_text ltx_font_bold" id="Ch0.S5.SS1.p1.1.1">(i) Reduced annotations for Diffusion Model.</span> The quality of synthetic data produced by a generative model typically depends heavily on the quantity and diversity of the paired training data used during the training phase <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib11" title="">chlap2021review </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib46" title="">jaipuria2020deflating </a></cite>. Nevertheless, the relationship between the number of annotated real tumors required for training the Diffusion Model and the performance of the Segmentation Model has not been extensively studied. DiffTumor found that the relationship between the amount of paired training data and the quality of synthetic data is not necessarily linear. Remarkably, it requires only one annotated tumor to train the Diffusion Model and to generate synthetic tumors for the subsequent training of the Segmentation Model. This finding is in contrast to the conventional wisdom in computer vision <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib81" title="">ramesh2022hierarchical </a></cite>, which often necessitates extensive datasets for training generative models. The results suggest that a smaller number of real tumors may suffice for training the Diffusion Model, particularly for early-stage tumors. Such a discovery could have profound implications for improving efficiency and reducing the costs associated with training generative models in the field of medical imaging.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S5.SS1.p2">
<p class="ltx_p" id="Ch0.S5.SS1.p2.3"><span class="ltx_text ltx_font_bold" id="Ch0.S5.SS1.p2.3.1">(ii) Accelerated tumor synthesis.</span>
The speed at which synthetic tumors are generated is critical for the practical use of synthetic data, particularly for accelerating the training of segmentation models. The synthesis speed of DiffTumor is significantly affected by the choice of timestep (<math alttext="T" class="ltx_Math" display="inline" id="Ch0.S5.SS1.p2.1.m1.1"><semantics id="Ch0.S5.SS1.p2.1.m1.1a"><mi id="Ch0.S5.SS1.p2.1.m1.1.1" xref="Ch0.S5.SS1.p2.1.m1.1.1.cmml">T</mi><annotation-xml encoding="MathML-Content" id="Ch0.S5.SS1.p2.1.m1.1b"><ci id="Ch0.S5.SS1.p2.1.m1.1.1.cmml" xref="Ch0.S5.SS1.p2.1.m1.1.1">𝑇</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S5.SS1.p2.1.m1.1c">T</annotation><annotation encoding="application/x-llamapun" id="Ch0.S5.SS1.p2.1.m1.1d">italic_T</annotation></semantics></math>). An investigation into the influence of timestep on the segmentation performance of the Segmentation Model has been conducted. It is observed that using DDPM <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib40" title="">ho2020denoising </a></cite> with one-step sampling, DiffTumor cannot synthesize realistic textures for both organ and tumor, which negatively affects the training of the Segmentation Model. Conversely, by setting <math alttext="T" class="ltx_Math" display="inline" id="Ch0.S5.SS1.p2.2.m2.1"><semantics id="Ch0.S5.SS1.p2.2.m2.1a"><mi id="Ch0.S5.SS1.p2.2.m2.1.1" xref="Ch0.S5.SS1.p2.2.m2.1.1.cmml">T</mi><annotation-xml encoding="MathML-Content" id="Ch0.S5.SS1.p2.2.m2.1b"><ci id="Ch0.S5.SS1.p2.2.m2.1.1.cmml" xref="Ch0.S5.SS1.p2.2.m2.1.1">𝑇</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S5.SS1.p2.2.m2.1c">T</annotation><annotation encoding="application/x-llamapun" id="Ch0.S5.SS1.p2.2.m2.1d">italic_T</annotation></semantics></math> higher than 1, DiffTumor can produce more realistic textures, leading to an enhanced performance of the segmentation model. Taking into account the balance between performance and synthesis efficiency, DiffTumor selects a timestep of <math alttext="T=4" class="ltx_Math" display="inline" id="Ch0.S5.SS1.p2.3.m3.1"><semantics id="Ch0.S5.SS1.p2.3.m3.1a"><mrow id="Ch0.S5.SS1.p2.3.m3.1.1" xref="Ch0.S5.SS1.p2.3.m3.1.1.cmml"><mi id="Ch0.S5.SS1.p2.3.m3.1.1.2" xref="Ch0.S5.SS1.p2.3.m3.1.1.2.cmml">T</mi><mo id="Ch0.S5.SS1.p2.3.m3.1.1.1" xref="Ch0.S5.SS1.p2.3.m3.1.1.1.cmml">=</mo><mn id="Ch0.S5.SS1.p2.3.m3.1.1.3" xref="Ch0.S5.SS1.p2.3.m3.1.1.3.cmml">4</mn></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S5.SS1.p2.3.m3.1b"><apply id="Ch0.S5.SS1.p2.3.m3.1.1.cmml" xref="Ch0.S5.SS1.p2.3.m3.1.1"><eq id="Ch0.S5.SS1.p2.3.m3.1.1.1.cmml" xref="Ch0.S5.SS1.p2.3.m3.1.1.1"></eq><ci id="Ch0.S5.SS1.p2.3.m3.1.1.2.cmml" xref="Ch0.S5.SS1.p2.3.m3.1.1.2">𝑇</ci><cn id="Ch0.S5.SS1.p2.3.m3.1.1.3.cmml" type="integer" xref="Ch0.S5.SS1.p2.3.m3.1.1.3">4</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S5.SS1.p2.3.m3.1c">T=4</annotation><annotation encoding="application/x-llamapun" id="Ch0.S5.SS1.p2.3.m3.1d">italic_T = 4</annotation></semantics></math> as the default setting for early tumor synthesis. This selection strikes a balance that allows for the generation of high-quality synthetic data while maintaining an acceptable level of efficiency.</p>
</div>
</section>
<section class="ltx_subsection" id="Ch0.S5.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">5.2 </span>Clinical Perspectives</h3>
<div class="ltx_para" id="Ch0.S5.SS2.p1">
<p class="ltx_p" id="Ch0.S5.SS2.p1.1">This learning-based approach can be widely applied because of the similar growth dynamics observed in tumors across various types and locations. Tumorigenesis is a complex, multistage process involving cellular and histological transformations, from precancerous lesions to malignant tumors. This progression, driven by genetic mutations and functional changes known as the ‘hallmarks of cancer,’ is consistent across tumor types <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib58" title="">lahouel2020revisiting </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib12" title="">choi2014ct </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib35" title="">hanahan2000hallmarks </a></cite>.
Early-stage tumors typically consist of well-to-moderately differentiated cells with mild atypia and invasiveness, showing rare hemorrhage and necrosis. They often appear as homogeneous nodules with slightly indistinct margins and small diameters in CT images <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib15" title="">chu2017diagnosis </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib90" title="">skarin2015atlas </a></cite>. In contrast, advanced tumors exhibit significant infiltration and destruction of surrounding tissues, extending beyond the original site and potentially affecting adjacent structures <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib5" title="">ayuso2018diagnosis </a></cite>. As tumors become more malignant, rapid growth leads to ischemia and necrosis due to insufficient vascular supply, resulting in heterogeneous patterns in CT images with features like hemorrhage and fibrosis <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib115" title="">yee2021tumor </a>; <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib84" title="">saar2008radiological </a></cite>. These characteristics are consistent across different populations, ages, and genders.</p>
</div>
</section>
</section>
<section class="ltx_section" id="Ch0.S6">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">6 </span>Tumor Synthesis Benchmark</h2>
<div class="ltx_para" id="Ch0.S6.p1">
<p class="ltx_p" id="Ch0.S6.p1.1">We evaluate the effectiveness of Pixel2Cancer and DiffTumor by comparing them with supervised models trained on real data and several prominent unsupervised anomaly segmentation methods. Table <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.T2" title="Table 2 ‣ 6 Tumor Synthesis Benchmark ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">2</span></a> highlights that synthetic data have significantly outperformed all these baseline methods, achieving a DSC of 59.77% and NSD of 61.29%. These results highlight the potential of synthetic strategies to avoid per-pixel manual annotation for tumor segmentation.</p>
</div>
<figure class="ltx_table" id="Ch0.T2">
<figcaption class="ltx_caption" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 2: </span>
<span class="ltx_text ltx_font_italic" id="Ch0.T2.7.1">Comparison with state-of-the-art unsupervised methods.</span> We compare the initial label-free modeling-based methods with other unsupervised anomaly segmentation baselines, tumor synthesis strategies, and fully-supervised methods. Modeling-based methods significantly outperform all other state-of-the-art unsupervised baseline methods and even surpass the fully-supervised method with detailed <span class="ltx_text ltx_font_italic" id="Ch0.T2.8.2">pixel-wise annotation</span>.
</figcaption><div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<table class="ltx_tabular ltx_figure_panel ltx_align_middle" id="Ch0.T2.9">
<tr class="ltx_tr" id="Ch0.T2.9.1">
<td class="ltx_td ltx_align_justify ltx_align_top" colspan="5" id="Ch0.T2.9.1.1"><span class="ltx_text ltx_font_italic" id="Ch0.T2.9.1.1.1" style="font-size:70%;">Liver tumor segmentation performance.</span></td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.1.2"></td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.9.2">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.2.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.2.1.1">
<span class="ltx_p" id="Ch0.T2.9.2.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.9.2.1.1.1.1" style="font-size:70%;">tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.2.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.2.2.1">
<span class="ltx_p" id="Ch0.T2.9.2.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.9.2.2.1.1.1" style="font-size:70%;">method</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.2.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.2.3.1">
<span class="ltx_p" id="Ch0.T2.9.2.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.9.2.3.1.1.1" style="font-size:70%;">architecture</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.2.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.2.4.1">
<span class="ltx_p" id="Ch0.T2.9.2.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.9.2.4.1.1.1" style="font-size:70%;">labeled / unlabeled CTs</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.2.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.2.5.1">
<span class="ltx_p" id="Ch0.T2.9.2.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.2.5.1.1.1" style="font-size:70%;">DSC (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.2.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.2.6.1">
<span class="ltx_p" id="Ch0.T2.9.2.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.2.6.1.1.1" style="font-size:70%;">NSD (%)</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.9.3">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.3.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.3.1.1">
<span class="ltx_p" id="Ch0.T2.9.3.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.9.3.1.1.1.1" style="font-size:70%;">none</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.3.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.3.2.1">
<span class="ltx_p" id="Ch0.T2.9.3.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.9.3.2.1.1.1" style="font-size:70%;">PatchCore </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib83" title="">roth2022towards </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.3.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.3.3.1">
<span class="ltx_p" id="Ch0.T2.9.3.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.9.3.3.1.1.1" style="font-size:70%;">Resnet50</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.3.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.3.4.1">
<span class="ltx_p" id="Ch0.T2.9.3.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.9.3.4.1.1.1" style="font-size:70%;">0 / 116</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.3.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.3.5.1">
<span class="ltx_p" id="Ch0.T2.9.3.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.3.5.1.1.1" style="font-size:70%;">15.9</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.3.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.3.6.1">
<span class="ltx_p" id="Ch0.T2.9.3.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.3.6.1.1.1" style="font-size:70%;">16.4</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.9.4">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.4.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.4.1.1">
<span class="ltx_p" id="Ch0.T2.9.4.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.9.4.1.1.1.1" style="font-size:70%;">none</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.4.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.4.2.1">
<span class="ltx_p" id="Ch0.T2.9.4.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.9.4.2.1.1.1" style="font-size:70%;">f-AnoGAN </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib86" title="">schlegl2019f </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.4.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.4.3.1">
<span class="ltx_p" id="Ch0.T2.9.4.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.9.4.3.1.1.1" style="font-size:70%;">Customized </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib7" title="">baur2021autoencoders </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.4.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.4.4.1">
<span class="ltx_p" id="Ch0.T2.9.4.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.9.4.4.1.1.1" style="font-size:70%;">0 / 116</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.4.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.4.5.1">
<span class="ltx_p" id="Ch0.T2.9.4.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.4.5.1.1.1" style="font-size:70%;">19.0</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.4.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.4.6.1">
<span class="ltx_p" id="Ch0.T2.9.4.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.4.6.1.1.1" style="font-size:70%;">16.9</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.9.5">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.5.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.5.1.1">
<span class="ltx_p" id="Ch0.T2.9.5.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.9.5.1.1.1.1" style="font-size:70%;">none</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.5.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.5.2.1">
<span class="ltx_p" id="Ch0.T2.9.5.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.9.5.2.1.1.1" style="font-size:70%;">VAE </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib53" title="">kingma2013auto </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.5.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.5.3.1">
<span class="ltx_p" id="Ch0.T2.9.5.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.9.5.3.1.1.1" style="font-size:70%;">Customized </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib7" title="">baur2021autoencoders </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.5.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.5.4.1">
<span class="ltx_p" id="Ch0.T2.9.5.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.9.5.4.1.1.1" style="font-size:70%;">0 / 116</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.5.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.5.5.1">
<span class="ltx_p" id="Ch0.T2.9.5.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.5.5.1.1.1" style="font-size:70%;">24.6</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.5.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.5.6.1">
<span class="ltx_p" id="Ch0.T2.9.5.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.5.6.1.1.1" style="font-size:70%;">23.6</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.9.6">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.6.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.6.1.1">
<span class="ltx_p" id="Ch0.T2.9.6.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.9.6.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.6.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.6.2.1">
<span class="ltx_p" id="Ch0.T2.9.6.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.9.6.2.1.1.1" style="font-size:70%;">Yao et al. </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib113" title="">yao2021label </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.6.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.6.3.1">
<span class="ltx_p" id="Ch0.T2.9.6.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.9.6.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.6.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.6.4.1">
<span class="ltx_p" id="Ch0.T2.9.6.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.9.6.4.1.1.1" style="font-size:70%;">0 / 116</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.6.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.6.5.1">
<span class="ltx_p" id="Ch0.T2.9.6.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.6.5.1.1.1" style="font-size:70%;">32.8</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.6.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.6.6.1">
<span class="ltx_p" id="Ch0.T2.9.6.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.6.6.1.1.1" style="font-size:70%;">31.3</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.9.7">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.7.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.7.1.1">
<span class="ltx_p" id="Ch0.T2.9.7.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.9.7.1.1.1.1" style="font-size:70%;">real</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.7.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.7.2.1">
<span class="ltx_p" id="Ch0.T2.9.7.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.9.7.2.1.1.1" style="font-size:70%;">fully-supervised</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.7.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.7.3.1">
<span class="ltx_p" id="Ch0.T2.9.7.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.9.7.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.7.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.7.4.1">
<span class="ltx_p" id="Ch0.T2.9.7.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.9.7.4.1.1.1" style="font-size:70%;">101 / 0</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.7.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.7.5.1">
<span class="ltx_p" id="Ch0.T2.9.7.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.7.5.1.1.1" style="font-size:70%;">56.7</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.9.7.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.7.6.1">
<span class="ltx_p" id="Ch0.T2.9.7.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.7.6.1.1.1" style="font-size:70%;">58.0</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.9.8">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.8.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.8.1.1">
<span class="ltx_p" id="Ch0.T2.9.8.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.9.8.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.8.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.8.2.1">
<span class="ltx_p" id="Ch0.T2.9.8.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.9.8.2.1.1.1" style="font-size:70%;">hand-crafted </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib42" title="">hu2023label </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.8.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.8.3.1">
<span class="ltx_p" id="Ch0.T2.9.8.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.9.8.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.8.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.8.4.1">
<span class="ltx_p" id="Ch0.T2.9.8.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.9.8.4.1.1.1" style="font-size:70%;">0 / 116</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.8.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.8.5.1">
<span class="ltx_p" id="Ch0.T2.9.8.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.8.5.1.1.1" style="font-size:70%;">59.8</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.8.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.8.6.1">
<span class="ltx_p" id="Ch0.T2.9.8.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.8.6.1.1.1" style="font-size:70%;">61.3</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.9.9">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.9.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.9.1.1">
<span class="ltx_p" id="Ch0.T2.9.9.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.9.9.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.9.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.9.2.1">
<span class="ltx_p" id="Ch0.T2.9.9.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.9.9.2.1.1.1" style="font-size:70%;">Pixel2Cancer </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib59" title="">lai2024pixel </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.9.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.9.3.1">
<span class="ltx_p" id="Ch0.T2.9.9.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.9.9.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.9.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.9.4.1">
<span class="ltx_p" id="Ch0.T2.9.9.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.9.9.4.1.1.1" style="font-size:70%;">0 / 116</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.9.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.9.5.1">
<span class="ltx_p" id="Ch0.T2.9.9.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.9.5.1.1.1" style="font-size:70%;">58.9</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.9.9.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.9.6.1">
<span class="ltx_p" id="Ch0.T2.9.9.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.9.6.1.1.1" style="font-size:70%;">63.7</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.9.10">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.9.10.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.10.1.1">
<span class="ltx_p" id="Ch0.T2.9.10.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.9.10.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.9.10.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.10.2.1">
<span class="ltx_p" id="Ch0.T2.9.10.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.9.10.2.1.1.1" style="font-size:70%;">DiffTumor </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib9" title="">chen2024towards </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.9.10.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.10.3.1">
<span class="ltx_p" id="Ch0.T2.9.10.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.9.10.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.9.10.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.10.4.1">
<span class="ltx_p" id="Ch0.T2.9.10.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.9.10.4.1.1.1" style="font-size:70%;">101 / 116</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.9.10.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.10.5.1">
<span class="ltx_p" id="Ch0.T2.9.10.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.10.5.1.1.1" style="font-size:70%;">70.9</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.9.10.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.9.10.6.1">
<span class="ltx_p" id="Ch0.T2.9.10.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.9.10.6.1.1.1" style="font-size:70%;">71.2</span></span>
</span>
</td>
</tr>
</table>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<table class="ltx_tabular ltx_figure_panel ltx_align_middle" id="Ch0.T2.10">
<tr class="ltx_tr" id="Ch0.T2.10.1">
<td class="ltx_td ltx_align_justify ltx_align_top" colspan="5" id="Ch0.T2.10.1.1"><span class="ltx_text ltx_font_italic" id="Ch0.T2.10.1.1.1" style="font-size:70%;">Pancreas tumor segmentation performance.</span></td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.1.2"></td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.10.2">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.2.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.2.1.1">
<span class="ltx_p" id="Ch0.T2.10.2.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.10.2.1.1.1.1" style="font-size:70%;">tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.2.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.2.2.1">
<span class="ltx_p" id="Ch0.T2.10.2.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.10.2.2.1.1.1" style="font-size:70%;">method</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.2.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.2.3.1">
<span class="ltx_p" id="Ch0.T2.10.2.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.10.2.3.1.1.1" style="font-size:70%;">architecture</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.2.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.2.4.1">
<span class="ltx_p" id="Ch0.T2.10.2.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.10.2.4.1.1.1" style="font-size:70%;">labeled / unlabeled CTs</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.2.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.2.5.1">
<span class="ltx_p" id="Ch0.T2.10.2.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.2.5.1.1.1" style="font-size:70%;">DSC (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.2.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.2.6.1">
<span class="ltx_p" id="Ch0.T2.10.2.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.2.6.1.1.1" style="font-size:70%;">NSD (%)</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.10.3">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.3.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.3.1.1">
<span class="ltx_p" id="Ch0.T2.10.3.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.10.3.1.1.1.1" style="font-size:70%;">real</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.3.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.3.2.1">
<span class="ltx_p" id="Ch0.T2.10.3.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.10.3.2.1.1.1" style="font-size:70%;">fully-supervised</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.3.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.3.3.1">
<span class="ltx_p" id="Ch0.T2.10.3.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.10.3.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.3.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.3.4.1">
<span class="ltx_p" id="Ch0.T2.10.3.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.10.3.4.1.1.1" style="font-size:70%;">96 / 0</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.3.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.3.5.1">
<span class="ltx_p" id="Ch0.T2.10.3.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.3.5.1.1.1" style="font-size:70%;">57.5</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.10.3.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.3.6.1">
<span class="ltx_p" id="Ch0.T2.10.3.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.3.6.1.1.1" style="font-size:70%;">56.5</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.10.4">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.4.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.4.1.1">
<span class="ltx_p" id="Ch0.T2.10.4.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.10.4.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.4.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.4.2.1">
<span class="ltx_p" id="Ch0.T2.10.4.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.10.4.2.1.1.1" style="font-size:70%;">hand-crafted </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib42" title="">hu2023label </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.4.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.4.3.1">
<span class="ltx_p" id="Ch0.T2.10.4.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.10.4.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.4.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.4.4.1">
<span class="ltx_p" id="Ch0.T2.10.4.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.10.4.4.1.1.1" style="font-size:70%;">0 / 104</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.4.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.4.5.1">
<span class="ltx_p" id="Ch0.T2.10.4.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.4.5.1.1.1" style="font-size:70%;">54.1</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.4.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.4.6.1">
<span class="ltx_p" id="Ch0.T2.10.4.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.4.6.1.1.1" style="font-size:70%;">52.2</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.10.5">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.5.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.5.1.1">
<span class="ltx_p" id="Ch0.T2.10.5.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.10.5.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.5.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.5.2.1">
<span class="ltx_p" id="Ch0.T2.10.5.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.10.5.2.1.1.1" style="font-size:70%;">Pixel2Cancer </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib59" title="">lai2024pixel </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.5.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.5.3.1">
<span class="ltx_p" id="Ch0.T2.10.5.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.10.5.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.5.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.5.4.1">
<span class="ltx_p" id="Ch0.T2.10.5.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.10.5.4.1.1.1" style="font-size:70%;">0 / 104</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.5.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.5.5.1">
<span class="ltx_p" id="Ch0.T2.10.5.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.5.5.1.1.1" style="font-size:70%;">60.9</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.10.5.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.5.6.1">
<span class="ltx_p" id="Ch0.T2.10.5.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.5.6.1.1.1" style="font-size:70%;">57.1</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.10.6">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.10.6.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.6.1.1">
<span class="ltx_p" id="Ch0.T2.10.6.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.10.6.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.10.6.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.6.2.1">
<span class="ltx_p" id="Ch0.T2.10.6.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.10.6.2.1.1.1" style="font-size:70%;">DiffTumor </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib9" title="">chen2024towards </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.10.6.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.6.3.1">
<span class="ltx_p" id="Ch0.T2.10.6.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.10.6.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.10.6.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.6.4.1">
<span class="ltx_p" id="Ch0.T2.10.6.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.10.6.4.1.1.1" style="font-size:70%;">96 / 104</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.10.6.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.6.5.1">
<span class="ltx_p" id="Ch0.T2.10.6.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.6.5.1.1.1" style="font-size:70%;">64.8</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.10.6.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.10.6.6.1">
<span class="ltx_p" id="Ch0.T2.10.6.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.10.6.6.1.1.1" style="font-size:70%;">60.5</span></span>
</span>
</td>
</tr>
</table>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<table class="ltx_tabular ltx_figure_panel ltx_align_middle" id="Ch0.T2.11">
<tr class="ltx_tr" id="Ch0.T2.11.1">
<td class="ltx_td ltx_align_justify ltx_align_top" colspan="5" id="Ch0.T2.11.1.1"><span class="ltx_text ltx_font_italic" id="Ch0.T2.11.1.1.1" style="font-size:70%;">Kidney tumor segmentation performance.</span></td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.1.2"></td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.11.2">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.2.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.2.1.1">
<span class="ltx_p" id="Ch0.T2.11.2.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.11.2.1.1.1.1" style="font-size:70%;">tumors</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.2.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.2.2.1">
<span class="ltx_p" id="Ch0.T2.11.2.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.11.2.2.1.1.1" style="font-size:70%;">method</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.2.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.2.3.1">
<span class="ltx_p" id="Ch0.T2.11.2.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.11.2.3.1.1.1" style="font-size:70%;">architecture</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.2.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.2.4.1">
<span class="ltx_p" id="Ch0.T2.11.2.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.11.2.4.1.1.1" style="font-size:70%;">labeled / unlabeled CTs</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.2.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.2.5.1">
<span class="ltx_p" id="Ch0.T2.11.2.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.2.5.1.1.1" style="font-size:70%;">DSC (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.2.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.2.6.1">
<span class="ltx_p" id="Ch0.T2.11.2.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.2.6.1.1.1" style="font-size:70%;">NSD (%)</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.11.3">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.3.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.3.1.1">
<span class="ltx_p" id="Ch0.T2.11.3.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.11.3.1.1.1.1" style="font-size:70%;">real</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.3.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.3.2.1">
<span class="ltx_p" id="Ch0.T2.11.3.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.11.3.2.1.1.1" style="font-size:70%;">fully-supervised</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.3.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.3.3.1">
<span class="ltx_p" id="Ch0.T2.11.3.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.11.3.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.3.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.3.4.1">
<span class="ltx_p" id="Ch0.T2.11.3.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.11.3.4.1.1.1" style="font-size:70%;">96 / 0</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.3.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.3.5.1">
<span class="ltx_p" id="Ch0.T2.11.3.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.3.5.1.1.1" style="font-size:70%;">71.3</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T2.11.3.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.3.6.1">
<span class="ltx_p" id="Ch0.T2.11.3.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.3.6.1.1.1" style="font-size:70%;">62.8</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.11.4">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.4.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.4.1.1">
<span class="ltx_p" id="Ch0.T2.11.4.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.11.4.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.4.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.4.2.1">
<span class="ltx_p" id="Ch0.T2.11.4.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.11.4.2.1.1.1" style="font-size:70%;">hand-crafted </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib42" title="">hu2023label </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.4.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.4.3.1">
<span class="ltx_p" id="Ch0.T2.11.4.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.11.4.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.4.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.4.4.1">
<span class="ltx_p" id="Ch0.T2.11.4.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.11.4.4.1.1.1" style="font-size:70%;">0 / 120</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.4.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.4.5.1">
<span class="ltx_p" id="Ch0.T2.11.4.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.4.5.1.1.1" style="font-size:70%;">63.2</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.4.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.4.6.1">
<span class="ltx_p" id="Ch0.T2.11.4.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.4.6.1.1.1" style="font-size:70%;">55.4</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.11.5">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.5.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.5.1.1">
<span class="ltx_p" id="Ch0.T2.11.5.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.11.5.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.5.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.5.2.1">
<span class="ltx_p" id="Ch0.T2.11.5.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.11.5.2.1.1.1" style="font-size:70%;">Pixel2Cancer </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib59" title="">lai2024pixel </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.5.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.5.3.1">
<span class="ltx_p" id="Ch0.T2.11.5.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.11.5.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.5.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.5.4.1">
<span class="ltx_p" id="Ch0.T2.11.5.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.11.5.4.1.1.1" style="font-size:70%;">0 / 120</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.5.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.5.5.1">
<span class="ltx_p" id="Ch0.T2.11.5.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.5.5.1.1.1" style="font-size:70%;">73.2</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T2.11.5.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.5.6.1">
<span class="ltx_p" id="Ch0.T2.11.5.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.5.6.1.1.1" style="font-size:70%;">65.0</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T2.11.6">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.11.6.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.6.1.1">
<span class="ltx_p" id="Ch0.T2.11.6.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T2.11.6.1.1.1.1" style="font-size:70%;">synt</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.11.6.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.6.2.1">
<span class="ltx_p" id="Ch0.T2.11.6.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T2.11.6.2.1.1.1" style="font-size:70%;">DiffTumor </span><cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib9" title="">chen2024towards </a></cite></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.11.6.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.6.3.1">
<span class="ltx_p" id="Ch0.T2.11.6.3.1.1" style="width:78.0pt;"><span class="ltx_text" id="Ch0.T2.11.6.3.1.1.1" style="font-size:70%;">U-Net</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.11.6.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.6.4.1">
<span class="ltx_p" id="Ch0.T2.11.6.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T2.11.6.4.1.1.1" style="font-size:70%;">96 / 120</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.11.6.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.6.5.1">
<span class="ltx_p" id="Ch0.T2.11.6.5.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.6.5.1.1.1" style="font-size:70%;">84.2</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_b" id="Ch0.T2.11.6.6">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T2.11.6.6.1">
<span class="ltx_p" id="Ch0.T2.11.6.6.1.1" style="width:52.0pt;"><span class="ltx_text" id="Ch0.T2.11.6.6.1.1.1" style="font-size:70%;">76.6</span></span>
</span>
</td>
</tr>
</table>
</div>
</div>
</figure>
<section class="ltx_subsection" id="Ch0.S6.SS1">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">6.1 </span>Case Study: Fake Tumors, Real Results</h3>
<div class="ltx_para ltx_noindent" id="Ch0.S6.SS1.p1">
<p class="ltx_p" id="Ch0.S6.SS1.p1.6"><span class="ltx_text ltx_font_bold" id="Ch0.S6.SS1.p1.6.1">Synthetic Liver Tumors.</span> In synthetic liver tumors, synthetic data have demonstrated significant superiority over real data across all stages, from small to large tumors. The hand-crafted approach proposed by <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib42" title="">hu2023label </a></cite> outperforms real data, achieving a <math alttext="2.3\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p1.1.m1.1"><semantics id="Ch0.S6.SS1.p1.1.m1.1a"><mrow id="Ch0.S6.SS1.p1.1.m1.1.1" xref="Ch0.S6.SS1.p1.1.m1.1.1.cmml"><mn id="Ch0.S6.SS1.p1.1.m1.1.1.2" xref="Ch0.S6.SS1.p1.1.m1.1.1.2.cmml">2.3</mn><mo id="Ch0.S6.SS1.p1.1.m1.1.1.1" xref="Ch0.S6.SS1.p1.1.m1.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p1.1.m1.1b"><apply id="Ch0.S6.SS1.p1.1.m1.1.1.cmml" xref="Ch0.S6.SS1.p1.1.m1.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p1.1.m1.1.1.1.cmml" xref="Ch0.S6.SS1.p1.1.m1.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p1.1.m1.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p1.1.m1.1.1.2">2.3</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p1.1.m1.1c">2.3\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p1.1.m1.1d">2.3 %</annotation></semantics></math> improvement in DSC and a <math alttext="3.3\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p1.2.m2.1"><semantics id="Ch0.S6.SS1.p1.2.m2.1a"><mrow id="Ch0.S6.SS1.p1.2.m2.1.1" xref="Ch0.S6.SS1.p1.2.m2.1.1.cmml"><mn id="Ch0.S6.SS1.p1.2.m2.1.1.2" xref="Ch0.S6.SS1.p1.2.m2.1.1.2.cmml">3.3</mn><mo id="Ch0.S6.SS1.p1.2.m2.1.1.1" xref="Ch0.S6.SS1.p1.2.m2.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p1.2.m2.1b"><apply id="Ch0.S6.SS1.p1.2.m2.1.1.cmml" xref="Ch0.S6.SS1.p1.2.m2.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p1.2.m2.1.1.1.cmml" xref="Ch0.S6.SS1.p1.2.m2.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p1.2.m2.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p1.2.m2.1.1.2">3.3</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p1.2.m2.1c">3.3\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p1.2.m2.1d">3.3 %</annotation></semantics></math> improvement in NSD. Pixel2Cancer has shown superior performance in liver segmentation, with a DSC improvement of <math alttext="2.2\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p1.3.m3.1"><semantics id="Ch0.S6.SS1.p1.3.m3.1a"><mrow id="Ch0.S6.SS1.p1.3.m3.1.1" xref="Ch0.S6.SS1.p1.3.m3.1.1.cmml"><mn id="Ch0.S6.SS1.p1.3.m3.1.1.2" xref="Ch0.S6.SS1.p1.3.m3.1.1.2.cmml">2.2</mn><mo id="Ch0.S6.SS1.p1.3.m3.1.1.1" xref="Ch0.S6.SS1.p1.3.m3.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p1.3.m3.1b"><apply id="Ch0.S6.SS1.p1.3.m3.1.1.cmml" xref="Ch0.S6.SS1.p1.3.m3.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p1.3.m3.1.1.1.cmml" xref="Ch0.S6.SS1.p1.3.m3.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p1.3.m3.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p1.3.m3.1.1.2">2.2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p1.3.m3.1c">2.2\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p1.3.m3.1d">2.2 %</annotation></semantics></math> and an NSD improvement of <math alttext="5.7\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p1.4.m4.1"><semantics id="Ch0.S6.SS1.p1.4.m4.1a"><mrow id="Ch0.S6.SS1.p1.4.m4.1.1" xref="Ch0.S6.SS1.p1.4.m4.1.1.cmml"><mn id="Ch0.S6.SS1.p1.4.m4.1.1.2" xref="Ch0.S6.SS1.p1.4.m4.1.1.2.cmml">5.7</mn><mo id="Ch0.S6.SS1.p1.4.m4.1.1.1" xref="Ch0.S6.SS1.p1.4.m4.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p1.4.m4.1b"><apply id="Ch0.S6.SS1.p1.4.m4.1.1.cmml" xref="Ch0.S6.SS1.p1.4.m4.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p1.4.m4.1.1.1.cmml" xref="Ch0.S6.SS1.p1.4.m4.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p1.4.m4.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p1.4.m4.1.1.2">5.7</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p1.4.m4.1c">5.7\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p1.4.m4.1d">5.7 %</annotation></semantics></math>. Additionally, DiffTumor has surpassed the performance of real data by <math alttext="4.0\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p1.5.m5.1"><semantics id="Ch0.S6.SS1.p1.5.m5.1a"><mrow id="Ch0.S6.SS1.p1.5.m5.1.1" xref="Ch0.S6.SS1.p1.5.m5.1.1.cmml"><mn id="Ch0.S6.SS1.p1.5.m5.1.1.2" xref="Ch0.S6.SS1.p1.5.m5.1.1.2.cmml">4.0</mn><mo id="Ch0.S6.SS1.p1.5.m5.1.1.1" xref="Ch0.S6.SS1.p1.5.m5.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p1.5.m5.1b"><apply id="Ch0.S6.SS1.p1.5.m5.1.1.cmml" xref="Ch0.S6.SS1.p1.5.m5.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p1.5.m5.1.1.1.cmml" xref="Ch0.S6.SS1.p1.5.m5.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p1.5.m5.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p1.5.m5.1.1.2">4.0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p1.5.m5.1c">4.0\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p1.5.m5.1d">4.0 %</annotation></semantics></math> in DSC and <math alttext="4.7\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p1.6.m6.1"><semantics id="Ch0.S6.SS1.p1.6.m6.1a"><mrow id="Ch0.S6.SS1.p1.6.m6.1.1" xref="Ch0.S6.SS1.p1.6.m6.1.1.cmml"><mn id="Ch0.S6.SS1.p1.6.m6.1.1.2" xref="Ch0.S6.SS1.p1.6.m6.1.1.2.cmml">4.7</mn><mo id="Ch0.S6.SS1.p1.6.m6.1.1.1" xref="Ch0.S6.SS1.p1.6.m6.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p1.6.m6.1b"><apply id="Ch0.S6.SS1.p1.6.m6.1.1.cmml" xref="Ch0.S6.SS1.p1.6.m6.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p1.6.m6.1.1.1.cmml" xref="Ch0.S6.SS1.p1.6.m6.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p1.6.m6.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p1.6.m6.1.1.2">4.7</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p1.6.m6.1c">4.7\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p1.6.m6.1d">4.7 %</annotation></semantics></math> in NSD.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S6.SS1.p2">
<p class="ltx_p" id="Ch0.S6.SS1.p2.4"><span class="ltx_text ltx_font_bold" id="Ch0.S6.SS1.p2.4.1">Synthetic Kidney Tumors.</span>
In kidney tumors, segmentation models have achieved superior performance when using synthetic data as data augmentation. Pixel2Cancer has shown superior performance in kidney segmentation, with a DSC improvement of <math alttext="2.4\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p2.1.m1.1"><semantics id="Ch0.S6.SS1.p2.1.m1.1a"><mrow id="Ch0.S6.SS1.p2.1.m1.1.1" xref="Ch0.S6.SS1.p2.1.m1.1.1.cmml"><mn id="Ch0.S6.SS1.p2.1.m1.1.1.2" xref="Ch0.S6.SS1.p2.1.m1.1.1.2.cmml">2.4</mn><mo id="Ch0.S6.SS1.p2.1.m1.1.1.1" xref="Ch0.S6.SS1.p2.1.m1.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p2.1.m1.1b"><apply id="Ch0.S6.SS1.p2.1.m1.1.1.cmml" xref="Ch0.S6.SS1.p2.1.m1.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p2.1.m1.1.1.1.cmml" xref="Ch0.S6.SS1.p2.1.m1.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p2.1.m1.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p2.1.m1.1.1.2">2.4</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p2.1.m1.1c">2.4\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p2.1.m1.1d">2.4 %</annotation></semantics></math> and an NSD improvement of <math alttext="3.2\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p2.2.m2.1"><semantics id="Ch0.S6.SS1.p2.2.m2.1a"><mrow id="Ch0.S6.SS1.p2.2.m2.1.1" xref="Ch0.S6.SS1.p2.2.m2.1.1.cmml"><mn id="Ch0.S6.SS1.p2.2.m2.1.1.2" xref="Ch0.S6.SS1.p2.2.m2.1.1.2.cmml">3.2</mn><mo id="Ch0.S6.SS1.p2.2.m2.1.1.1" xref="Ch0.S6.SS1.p2.2.m2.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p2.2.m2.1b"><apply id="Ch0.S6.SS1.p2.2.m2.1.1.cmml" xref="Ch0.S6.SS1.p2.2.m2.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p2.2.m2.1.1.1.cmml" xref="Ch0.S6.SS1.p2.2.m2.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p2.2.m2.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p2.2.m2.1.1.2">3.2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p2.2.m2.1c">3.2\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p2.2.m2.1d">3.2 %</annotation></semantics></math>. Additionally, DiffTumor has surpassed the performance of real data by <math alttext="7.0\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p2.3.m3.1"><semantics id="Ch0.S6.SS1.p2.3.m3.1a"><mrow id="Ch0.S6.SS1.p2.3.m3.1.1" xref="Ch0.S6.SS1.p2.3.m3.1.1.cmml"><mn id="Ch0.S6.SS1.p2.3.m3.1.1.2" xref="Ch0.S6.SS1.p2.3.m3.1.1.2.cmml">7.0</mn><mo id="Ch0.S6.SS1.p2.3.m3.1.1.1" xref="Ch0.S6.SS1.p2.3.m3.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p2.3.m3.1b"><apply id="Ch0.S6.SS1.p2.3.m3.1.1.cmml" xref="Ch0.S6.SS1.p2.3.m3.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p2.3.m3.1.1.1.cmml" xref="Ch0.S6.SS1.p2.3.m3.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p2.3.m3.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p2.3.m3.1.1.2">7.0</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p2.3.m3.1c">7.0\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p2.3.m3.1d">7.0 %</annotation></semantics></math> in DSC and <math alttext="6.7\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p2.4.m4.1"><semantics id="Ch0.S6.SS1.p2.4.m4.1a"><mrow id="Ch0.S6.SS1.p2.4.m4.1.1" xref="Ch0.S6.SS1.p2.4.m4.1.1.cmml"><mn id="Ch0.S6.SS1.p2.4.m4.1.1.2" xref="Ch0.S6.SS1.p2.4.m4.1.1.2.cmml">6.7</mn><mo id="Ch0.S6.SS1.p2.4.m4.1.1.1" xref="Ch0.S6.SS1.p2.4.m4.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p2.4.m4.1b"><apply id="Ch0.S6.SS1.p2.4.m4.1.1.cmml" xref="Ch0.S6.SS1.p2.4.m4.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p2.4.m4.1.1.1.cmml" xref="Ch0.S6.SS1.p2.4.m4.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p2.4.m4.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p2.4.m4.1.1.2">6.7</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p2.4.m4.1c">6.7\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p2.4.m4.1d">6.7 %</annotation></semantics></math> in NSD.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S6.SS1.p3">
<p class="ltx_p" id="Ch0.S6.SS1.p3.4"><span class="ltx_text ltx_font_bold" id="Ch0.S6.SS1.p3.4.1">Synthetic Pancreatic Tumors.</span>
Segmentation models also have achieved superior performance in pancreatic tumor segmentation when using synthetic data as data augmentation. Pixel2Cancer has shown superior performance in pancreas segmentation, with a DSC improvement of <math alttext="3.9\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p3.1.m1.1"><semantics id="Ch0.S6.SS1.p3.1.m1.1a"><mrow id="Ch0.S6.SS1.p3.1.m1.1.1" xref="Ch0.S6.SS1.p3.1.m1.1.1.cmml"><mn id="Ch0.S6.SS1.p3.1.m1.1.1.2" xref="Ch0.S6.SS1.p3.1.m1.1.1.2.cmml">3.9</mn><mo id="Ch0.S6.SS1.p3.1.m1.1.1.1" xref="Ch0.S6.SS1.p3.1.m1.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p3.1.m1.1b"><apply id="Ch0.S6.SS1.p3.1.m1.1.1.cmml" xref="Ch0.S6.SS1.p3.1.m1.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p3.1.m1.1.1.1.cmml" xref="Ch0.S6.SS1.p3.1.m1.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p3.1.m1.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p3.1.m1.1.1.2">3.9</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p3.1.m1.1c">3.9\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p3.1.m1.1d">3.9 %</annotation></semantics></math> and an NSD improvement of <math alttext="1.9\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p3.2.m2.1"><semantics id="Ch0.S6.SS1.p3.2.m2.1a"><mrow id="Ch0.S6.SS1.p3.2.m2.1.1" xref="Ch0.S6.SS1.p3.2.m2.1.1.cmml"><mn id="Ch0.S6.SS1.p3.2.m2.1.1.2" xref="Ch0.S6.SS1.p3.2.m2.1.1.2.cmml">1.9</mn><mo id="Ch0.S6.SS1.p3.2.m2.1.1.1" xref="Ch0.S6.SS1.p3.2.m2.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p3.2.m2.1b"><apply id="Ch0.S6.SS1.p3.2.m2.1.1.cmml" xref="Ch0.S6.SS1.p3.2.m2.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p3.2.m2.1.1.1.cmml" xref="Ch0.S6.SS1.p3.2.m2.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p3.2.m2.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p3.2.m2.1.1.2">1.9</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p3.2.m2.1c">1.9\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p3.2.m2.1d">1.9 %</annotation></semantics></math>. DiffTumor has surpassed the performance of real data by <math alttext="8.2\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p3.3.m3.1"><semantics id="Ch0.S6.SS1.p3.3.m3.1a"><mrow id="Ch0.S6.SS1.p3.3.m3.1.1" xref="Ch0.S6.SS1.p3.3.m3.1.1.cmml"><mn id="Ch0.S6.SS1.p3.3.m3.1.1.2" xref="Ch0.S6.SS1.p3.3.m3.1.1.2.cmml">8.2</mn><mo id="Ch0.S6.SS1.p3.3.m3.1.1.1" xref="Ch0.S6.SS1.p3.3.m3.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p3.3.m3.1b"><apply id="Ch0.S6.SS1.p3.3.m3.1.1.cmml" xref="Ch0.S6.SS1.p3.3.m3.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p3.3.m3.1.1.1.cmml" xref="Ch0.S6.SS1.p3.3.m3.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p3.3.m3.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p3.3.m3.1.1.2">8.2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p3.3.m3.1c">8.2\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p3.3.m3.1d">8.2 %</annotation></semantics></math> in DSC and <math alttext="9.4\%" class="ltx_Math" display="inline" id="Ch0.S6.SS1.p3.4.m4.1"><semantics id="Ch0.S6.SS1.p3.4.m4.1a"><mrow id="Ch0.S6.SS1.p3.4.m4.1.1" xref="Ch0.S6.SS1.p3.4.m4.1.1.cmml"><mn id="Ch0.S6.SS1.p3.4.m4.1.1.2" xref="Ch0.S6.SS1.p3.4.m4.1.1.2.cmml">9.4</mn><mo id="Ch0.S6.SS1.p3.4.m4.1.1.1" xref="Ch0.S6.SS1.p3.4.m4.1.1.1.cmml">%</mo></mrow><annotation-xml encoding="MathML-Content" id="Ch0.S6.SS1.p3.4.m4.1b"><apply id="Ch0.S6.SS1.p3.4.m4.1.1.cmml" xref="Ch0.S6.SS1.p3.4.m4.1.1"><csymbol cd="latexml" id="Ch0.S6.SS1.p3.4.m4.1.1.1.cmml" xref="Ch0.S6.SS1.p3.4.m4.1.1.1">percent</csymbol><cn id="Ch0.S6.SS1.p3.4.m4.1.1.2.cmml" type="float" xref="Ch0.S6.SS1.p3.4.m4.1.1.2">9.4</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.SS1.p3.4.m4.1c">9.4\%</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.SS1.p3.4.m4.1d">9.4 %</annotation></semantics></math> in NSD.</p>
</div>
<figure class="ltx_figure" id="Ch0.F9"><img alt="Refer to caption" class="ltx_graphics ltx_centering ltx_img_landscape" height="337" id="Ch0.F9.g1" src="extracted/5843742/fig_CT_synthetic_tumor.jpg" width="598"/>
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 9: </span>Visual Turing Test. Can you find some examples of synthetic data in the CT images? Answers are in <a class="ltx_ref ltx_href ltx_font_italic" href="https://hub.jhu.edu/2024/05/30/researchers-create-artificial-tumors-for-cancer-research/" title="">Johns Hopkins Researchers Create Artificial Tumors to Help AI Detect Early-Stage Cancer</a>.</figcaption>
</figure>
<figure class="ltx_table" id="Ch0.T3">
<figcaption class="ltx_caption" style="font-size:70%;"><span class="ltx_tag ltx_tag_table">Table 3: </span>
<span class="ltx_text ltx_font_italic" id="Ch0.T3.15.1">Results of reader study.</span> Pixel2Cancer (top table): The test was conducted with three medical professionals having 7, 9, and 14 years of experience, respectively. Each professional evaluated 50 CT images for each organ, consisting of both real and synthetic tumors. They were tasked with categorizing each CT image as either <span class="ltx_text ltx_font_italic" id="Ch0.T3.16.2">real</span>, <span class="ltx_text ltx_font_italic" id="Ch0.T3.17.3">synthetic</span>, or <span class="ltx_text ltx_font_italic" id="Ch0.T3.18.4">unsure</span>. DiffTumor (bottom table): Visual Turing test over three organs has been conducted with two radiologists (R1 and R2). Both radiologists are provided with 60 three-dimensional CT volumes of each organ, including 30 scans with real tumors and the remaining 30 with synthetic ones. Radiologists are tasked to label each CT volume as <span class="ltx_text ltx_font_italic" id="Ch0.T3.19.5">real</span> or <span class="ltx_text ltx_font_italic" id="Ch0.T3.20.6">synthetic</span>. A lower specificity score indicates a higher number of synthetic tumors being identified as real.
</figcaption><div class="ltx_flex_figure">
<div class="ltx_flex_cell ltx_flex_size_1">
<table class="ltx_tabular ltx_figure_panel ltx_align_middle" id="Ch0.T3.21">
<tr class="ltx_tr" id="Ch0.T3.21.1">
<td class="ltx_td ltx_align_justify ltx_align_top" colspan="5" id="Ch0.T3.21.1.1"><span class="ltx_text ltx_font_italic" id="Ch0.T3.21.1.1.1" style="font-size:70%;">Modeling-based Approach: Pixel2Cancer</span></td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.2">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.21.2.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.2.1.1">
<span class="ltx_p" id="Ch0.T3.21.2.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T3.21.2.1.1.1.1" style="font-size:70%;">reader</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.21.2.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.2.2.1">
<span class="ltx_p" id="Ch0.T3.21.2.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.2.2.1.1.1" style="font-size:70%;">metric</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.21.2.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.2.3.1">
<span class="ltx_p" id="Ch0.T3.21.2.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.2.3.1.1.1" style="font-size:70%;">liver</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.21.2.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.2.4.1">
<span class="ltx_p" id="Ch0.T3.21.2.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.2.4.1.1.1" style="font-size:70%;">pancreas</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.21.2.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.2.5.1">
<span class="ltx_p" id="Ch0.T3.21.2.5.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.2.5.1.1.1" style="font-size:70%;">kidneys</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.3">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.3.1" rowspan="3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.3.1.1">
<span class="ltx_p" id="Ch0.T3.21.3.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T3.21.3.1.1.1.1" style="font-size:70%;"><span class="ltx_text" id="Ch0.T3.21.3.1.1.1.1.1"></span><span class="ltx_text" id="Ch0.T3.21.3.1.1.1.1.2">
<span class="ltx_tabular ltx_align_middle" id="Ch0.T3.21.3.1.1.1.1.2.1">
<span class="ltx_tr" id="Ch0.T3.21.3.1.1.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T3.21.3.1.1.1.1.2.1.1.1">R1</span></span>
</span></span> <span class="ltx_text" id="Ch0.T3.21.3.1.1.1.1.3"></span></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.3.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.3.2.1">
<span class="ltx_p" id="Ch0.T3.21.3.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.3.2.1.1.1" style="font-size:70%;">sensitivity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.3.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.3.3.1">
<span class="ltx_p" id="Ch0.T3.21.3.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.3.3.1.1.1" style="font-size:70%;">100</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.3.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.3.4.1">
<span class="ltx_p" id="Ch0.T3.21.3.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.3.4.1.1.1" style="font-size:70%;">95.0</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.3.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.3.5.1">
<span class="ltx_p" id="Ch0.T3.21.3.5.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.3.5.1.1.1" style="font-size:70%;">95.5</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.4">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.4.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.4.1.1">
<span class="ltx_p" id="Ch0.T3.21.4.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.4.1.1.1.1" style="font-size:70%;">specificity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.4.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.4.2.1">
<span class="ltx_p" id="Ch0.T3.21.4.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.4.2.1.1.1" style="font-size:70%;">27.3</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.4.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.4.3.1">
<span class="ltx_p" id="Ch0.T3.21.4.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.4.3.1.1.1" style="font-size:70%;">22.7</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.4.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.4.4.1">
<span class="ltx_p" id="Ch0.T3.21.4.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.4.4.1.1.1" style="font-size:70%;">26.7</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.5">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.5.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.5.1.1">
<span class="ltx_p" id="Ch0.T3.21.5.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.5.1.1.1.1" style="font-size:70%;">accuracy (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.5.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.5.2.1">
<span class="ltx_p" id="Ch0.T3.21.5.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.5.2.1.1.1" style="font-size:70%;">60.9</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.5.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.5.3.1">
<span class="ltx_p" id="Ch0.T3.21.5.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.5.3.1.1.1" style="font-size:70%;">57.1</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.5.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.5.4.1">
<span class="ltx_p" id="Ch0.T3.21.5.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.5.4.1.1.1" style="font-size:70%;">67.6</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.6">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.6.1" rowspan="3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.6.1.1">
<span class="ltx_p" id="Ch0.T3.21.6.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T3.21.6.1.1.1.1" style="font-size:70%;"><span class="ltx_text" id="Ch0.T3.21.6.1.1.1.1.1"></span><span class="ltx_text" id="Ch0.T3.21.6.1.1.1.1.2">
<span class="ltx_tabular ltx_align_middle" id="Ch0.T3.21.6.1.1.1.1.2.1">
<span class="ltx_tr" id="Ch0.T3.21.6.1.1.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T3.21.6.1.1.1.1.2.1.1.1">R2</span></span>
</span></span> <span class="ltx_text" id="Ch0.T3.21.6.1.1.1.1.3"></span></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.6.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.6.2.1">
<span class="ltx_p" id="Ch0.T3.21.6.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.6.2.1.1.1" style="font-size:70%;">sensitivity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.6.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.6.3.1">
<span class="ltx_p" id="Ch0.T3.21.6.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.6.3.1.1.1" style="font-size:70%;">94.7</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.6.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.6.4.1">
<span class="ltx_p" id="Ch0.T3.21.6.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.6.4.1.1.1" style="font-size:70%;">87.5</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.6.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.6.5.1">
<span class="ltx_p" id="Ch0.T3.21.6.5.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.6.5.1.1.1" style="font-size:70%;">90.0</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.7">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.7.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.7.1.1">
<span class="ltx_p" id="Ch0.T3.21.7.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.7.1.1.1.1" style="font-size:70%;">specificity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.7.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.7.2.1">
<span class="ltx_p" id="Ch0.T3.21.7.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.7.2.1.1.1" style="font-size:70%;">47.8</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.7.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.7.3.1">
<span class="ltx_p" id="Ch0.T3.21.7.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.7.3.1.1.1" style="font-size:70%;">47.4</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.7.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.7.4.1">
<span class="ltx_p" id="Ch0.T3.21.7.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.7.4.1.1.1" style="font-size:70%;">56.3</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.8">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.8.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.8.1.1">
<span class="ltx_p" id="Ch0.T3.21.8.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.8.1.1.1.1" style="font-size:70%;">accuracy (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.8.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.8.2.1">
<span class="ltx_p" id="Ch0.T3.21.8.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.8.2.1.1.1" style="font-size:70%;">69.1</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.8.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.8.3.1">
<span class="ltx_p" id="Ch0.T3.21.8.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.8.3.1.1.1" style="font-size:70%;">65.7</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.8.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.8.4.1">
<span class="ltx_p" id="Ch0.T3.21.8.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.8.4.1.1.1" style="font-size:70%;">75.0</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.9">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb ltx_border_t" id="Ch0.T3.21.9.1" rowspan="3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.9.1.1">
<span class="ltx_p" id="Ch0.T3.21.9.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T3.21.9.1.1.1.1" style="font-size:70%;"><span class="ltx_text" id="Ch0.T3.21.9.1.1.1.1.1"></span><span class="ltx_text" id="Ch0.T3.21.9.1.1.1.1.2">
<span class="ltx_tabular ltx_align_middle" id="Ch0.T3.21.9.1.1.1.1.2.1">
<span class="ltx_tr" id="Ch0.T3.21.9.1.1.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T3.21.9.1.1.1.1.2.1.1.1">R3</span></span>
</span></span> <span class="ltx_text" id="Ch0.T3.21.9.1.1.1.1.3"></span></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.9.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.9.2.1">
<span class="ltx_p" id="Ch0.T3.21.9.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.9.2.1.1.1" style="font-size:70%;">sensitivity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.9.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.9.3.1">
<span class="ltx_p" id="Ch0.T3.21.9.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.9.3.1.1.1" style="font-size:70%;">100</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.9.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.9.4.1">
<span class="ltx_p" id="Ch0.T3.21.9.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.9.4.1.1.1" style="font-size:70%;">100</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.21.9.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.9.5.1">
<span class="ltx_p" id="Ch0.T3.21.9.5.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.9.5.1.1.1" style="font-size:70%;">100</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.10">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.10.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.10.1.1">
<span class="ltx_p" id="Ch0.T3.21.10.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.10.1.1.1.1" style="font-size:70%;">specificity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.10.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.10.2.1">
<span class="ltx_p" id="Ch0.T3.21.10.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.10.2.1.1.1" style="font-size:70%;">45.4</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.10.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.10.3.1">
<span class="ltx_p" id="Ch0.T3.21.10.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.10.3.1.1.1" style="font-size:70%;">55.6</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.21.10.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.10.4.1">
<span class="ltx_p" id="Ch0.T3.21.10.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.10.4.1.1.1" style="font-size:70%;">57.9</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.21.11">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T3.21.11.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.11.1.1">
<span class="ltx_p" id="Ch0.T3.21.11.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.21.11.1.1.1.1" style="font-size:70%;">accuracy (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T3.21.11.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.11.2.1">
<span class="ltx_p" id="Ch0.T3.21.11.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.11.2.1.1.1" style="font-size:70%;">68.4</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T3.21.11.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.11.3.1">
<span class="ltx_p" id="Ch0.T3.21.11.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.11.3.1.1.1" style="font-size:70%;">72.4</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T3.21.11.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.21.11.4.1">
<span class="ltx_p" id="Ch0.T3.21.11.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.21.11.4.1.1.1" style="font-size:70%;">75.8</span></span>
</span>
</td>
</tr>
</table>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<ul class="ltx_itemize ltx_figure_panel" id="Ch0.S6.I1">
<li class="ltx_item" id="Ch0.S6.I1.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S6.I1.i1.p1">
<p class="ltx_p" id="Ch0.S6.I1.i1.p1.2"><span class="ltx_text" id="Ch0.S6.I1.i1.p1.2.1" style="font-size:70%;">positives: real tumors (</span><math alttext="N" class="ltx_Math" display="inline" id="Ch0.S6.I1.i1.p1.1.m1.1"><semantics id="Ch0.S6.I1.i1.p1.1.m1.1a"><mi id="Ch0.S6.I1.i1.p1.1.m1.1.1" mathsize="70%" xref="Ch0.S6.I1.i1.p1.1.m1.1.1.cmml">N</mi><annotation-xml encoding="MathML-Content" id="Ch0.S6.I1.i1.p1.1.m1.1b"><ci id="Ch0.S6.I1.i1.p1.1.m1.1.1.cmml" xref="Ch0.S6.I1.i1.p1.1.m1.1.1">𝑁</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.I1.i1.p1.1.m1.1c">N</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.I1.i1.p1.1.m1.1d">italic_N</annotation></semantics></math><span class="ltx_text" id="Ch0.S6.I1.i1.p1.2.2" style="font-size:70%;"> = 25); negatives: synthetic tumors (</span><math alttext="N" class="ltx_Math" display="inline" id="Ch0.S6.I1.i1.p1.2.m2.1"><semantics id="Ch0.S6.I1.i1.p1.2.m2.1a"><mi id="Ch0.S6.I1.i1.p1.2.m2.1.1" mathsize="70%" xref="Ch0.S6.I1.i1.p1.2.m2.1.1.cmml">N</mi><annotation-xml encoding="MathML-Content" id="Ch0.S6.I1.i1.p1.2.m2.1b"><ci id="Ch0.S6.I1.i1.p1.2.m2.1.1.cmml" xref="Ch0.S6.I1.i1.p1.2.m2.1.1">𝑁</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.I1.i1.p1.2.m2.1c">N</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.I1.i1.p1.2.m2.1d">italic_N</annotation></semantics></math><span class="ltx_text" id="Ch0.S6.I1.i1.p1.2.3" style="font-size:70%;"> = 25).</span></p>
</div>
</li>
</ul>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<table class="ltx_tabular ltx_figure_panel ltx_align_middle" id="Ch0.T3.22">
<tr class="ltx_tr" id="Ch0.T3.22.1">
<td class="ltx_td ltx_align_justify ltx_align_top" colspan="5" id="Ch0.T3.22.1.1"><span class="ltx_text ltx_font_italic" id="Ch0.T3.22.1.1.1" style="font-size:70%;">Learning-based Approach: DiffTumor</span></td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.22.2">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.22.2.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.2.1.1">
<span class="ltx_p" id="Ch0.T3.22.2.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T3.22.2.1.1.1.1" style="font-size:70%;">reader</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.22.2.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.2.2.1">
<span class="ltx_p" id="Ch0.T3.22.2.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.22.2.2.1.1.1" style="font-size:70%;">metric</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.22.2.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.2.3.1">
<span class="ltx_p" id="Ch0.T3.22.2.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.2.3.1.1.1" style="font-size:70%;">liver</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.22.2.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.2.4.1">
<span class="ltx_p" id="Ch0.T3.22.2.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.2.4.1.1.1" style="font-size:70%;">pancreas</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_tt" id="Ch0.T3.22.2.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.2.5.1">
<span class="ltx_p" id="Ch0.T3.22.2.5.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.2.5.1.1.1" style="font-size:70%;">kidneys</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.22.3">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.22.3.1" rowspan="3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.3.1.1">
<span class="ltx_p" id="Ch0.T3.22.3.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T3.22.3.1.1.1.1" style="font-size:70%;"><span class="ltx_text" id="Ch0.T3.22.3.1.1.1.1.1"></span><span class="ltx_text" id="Ch0.T3.22.3.1.1.1.1.2">
<span class="ltx_tabular ltx_align_middle" id="Ch0.T3.22.3.1.1.1.1.2.1">
<span class="ltx_tr" id="Ch0.T3.22.3.1.1.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T3.22.3.1.1.1.1.2.1.1.1">R1</span></span>
</span></span> <span class="ltx_text" id="Ch0.T3.22.3.1.1.1.1.3"></span></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.22.3.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.3.2.1">
<span class="ltx_p" id="Ch0.T3.22.3.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.22.3.2.1.1.1" style="font-size:70%;">sensitivity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.22.3.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.3.3.1">
<span class="ltx_p" id="Ch0.T3.22.3.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.3.3.1.1.1" style="font-size:70%;">100</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.22.3.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.3.4.1">
<span class="ltx_p" id="Ch0.T3.22.3.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.3.4.1.1.1" style="font-size:70%;">97.1</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.22.3.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.3.5.1">
<span class="ltx_p" id="Ch0.T3.22.3.5.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.3.5.1.1.1" style="font-size:70%;">92.9</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.22.4">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.4.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.4.1.1">
<span class="ltx_p" id="Ch0.T3.22.4.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.22.4.1.1.1.1" style="font-size:70%;">specificity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.4.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.4.2.1">
<span class="ltx_p" id="Ch0.T3.22.4.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.4.2.1.1.1" style="font-size:70%;">2.9</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.4.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.4.3.1">
<span class="ltx_p" id="Ch0.T3.22.4.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.4.3.1.1.1" style="font-size:70%;">0.0</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.4.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.4.4.1">
<span class="ltx_p" id="Ch0.T3.22.4.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.4.4.1.1.1" style="font-size:70%;">3.1</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.22.5">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.5.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.5.1.1">
<span class="ltx_p" id="Ch0.T3.22.5.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.22.5.1.1.1.1" style="font-size:70%;">accuracy (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.5.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.5.2.1">
<span class="ltx_p" id="Ch0.T3.22.5.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.5.2.1.1.1" style="font-size:70%;">45.0</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.5.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.5.3.1">
<span class="ltx_p" id="Ch0.T3.22.5.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.5.3.1.1.1" style="font-size:70%;">56.7</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.5.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.5.4.1">
<span class="ltx_p" id="Ch0.T3.22.5.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.5.4.1.1.1" style="font-size:70%;">45.0</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.22.6">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb ltx_border_t" id="Ch0.T3.22.6.1" rowspan="3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.6.1.1">
<span class="ltx_p" id="Ch0.T3.22.6.1.1.1" style="width:43.4pt;"><span class="ltx_text" id="Ch0.T3.22.6.1.1.1.1" style="font-size:70%;"><span class="ltx_text" id="Ch0.T3.22.6.1.1.1.1.1"></span><span class="ltx_text" id="Ch0.T3.22.6.1.1.1.1.2">
<span class="ltx_tabular ltx_align_middle" id="Ch0.T3.22.6.1.1.1.1.2.1">
<span class="ltx_tr" id="Ch0.T3.22.6.1.1.1.1.2.1.1">
<span class="ltx_td ltx_nopad_r ltx_align_left" id="Ch0.T3.22.6.1.1.1.1.2.1.1.1">R2</span></span>
</span></span> <span class="ltx_text" id="Ch0.T3.22.6.1.1.1.1.3"></span></span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.22.6.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.6.2.1">
<span class="ltx_p" id="Ch0.T3.22.6.2.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.22.6.2.1.1.1" style="font-size:70%;">sensitivity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.22.6.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.6.3.1">
<span class="ltx_p" id="Ch0.T3.22.6.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.6.3.1.1.1" style="font-size:70%;">84.6</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.22.6.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.6.4.1">
<span class="ltx_p" id="Ch0.T3.22.6.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.6.4.1.1.1" style="font-size:70%;">100</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_t" id="Ch0.T3.22.6.5">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.6.5.1">
<span class="ltx_p" id="Ch0.T3.22.6.5.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.6.5.1.1.1" style="font-size:70%;">100</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.22.7">
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.7.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.7.1.1">
<span class="ltx_p" id="Ch0.T3.22.7.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.22.7.1.1.1.1" style="font-size:70%;">specificity (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.7.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.7.2.1">
<span class="ltx_p" id="Ch0.T3.22.7.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.7.2.1.1.1" style="font-size:70%;">47.1</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.7.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.7.3.1">
<span class="ltx_p" id="Ch0.T3.22.7.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.7.3.1.1.1" style="font-size:70%;">44.0</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top" id="Ch0.T3.22.7.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.7.4.1">
<span class="ltx_p" id="Ch0.T3.22.7.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.7.4.1.1.1" style="font-size:70%;">65.6</span></span>
</span>
</td>
</tr>
<tr class="ltx_tr" id="Ch0.T3.22.8">
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T3.22.8.1">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.8.1.1">
<span class="ltx_p" id="Ch0.T3.22.8.1.1.1" style="width:86.7pt;"><span class="ltx_text" id="Ch0.T3.22.8.1.1.1.1" style="font-size:70%;">accuracy (%)</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T3.22.8.2">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.8.2.1">
<span class="ltx_p" id="Ch0.T3.22.8.2.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.8.2.1.1.1" style="font-size:70%;">63.3</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T3.22.8.3">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.8.3.1">
<span class="ltx_p" id="Ch0.T3.22.8.3.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.8.3.1.1.1" style="font-size:70%;">76.7</span></span>
</span>
</td>
<td class="ltx_td ltx_align_justify ltx_align_top ltx_border_bb" id="Ch0.T3.22.8.4">
<span class="ltx_inline-block ltx_align_top" id="Ch0.T3.22.8.4.1">
<span class="ltx_p" id="Ch0.T3.22.8.4.1.1" style="width:95.4pt;"><span class="ltx_text" id="Ch0.T3.22.8.4.1.1.1" style="font-size:70%;">81.7</span></span>
</span>
</td>
</tr>
</table>
</div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<ul class="ltx_itemize ltx_figure_panel" id="Ch0.S6.I2">
<li class="ltx_item" id="Ch0.S6.I2.i1" style="list-style-type:none;">
<span class="ltx_tag ltx_tag_item">•</span>
<div class="ltx_para" id="Ch0.S6.I2.i1.p1">
<p class="ltx_p" id="Ch0.S6.I2.i1.p1.2"><span class="ltx_text" id="Ch0.S6.I2.i1.p1.2.1" style="font-size:70%;">positives: real tumors (</span><math alttext="N" class="ltx_Math" display="inline" id="Ch0.S6.I2.i1.p1.1.m1.1"><semantics id="Ch0.S6.I2.i1.p1.1.m1.1a"><mi id="Ch0.S6.I2.i1.p1.1.m1.1.1" mathsize="70%" xref="Ch0.S6.I2.i1.p1.1.m1.1.1.cmml">N</mi><annotation-xml encoding="MathML-Content" id="Ch0.S6.I2.i1.p1.1.m1.1b"><ci id="Ch0.S6.I2.i1.p1.1.m1.1.1.cmml" xref="Ch0.S6.I2.i1.p1.1.m1.1.1">𝑁</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.I2.i1.p1.1.m1.1c">N</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.I2.i1.p1.1.m1.1d">italic_N</annotation></semantics></math><span class="ltx_text" id="Ch0.S6.I2.i1.p1.2.2" style="font-size:70%;"> = 30); negatives: synthetic tumors (</span><math alttext="N" class="ltx_Math" display="inline" id="Ch0.S6.I2.i1.p1.2.m2.1"><semantics id="Ch0.S6.I2.i1.p1.2.m2.1a"><mi id="Ch0.S6.I2.i1.p1.2.m2.1.1" mathsize="70%" xref="Ch0.S6.I2.i1.p1.2.m2.1.1.cmml">N</mi><annotation-xml encoding="MathML-Content" id="Ch0.S6.I2.i1.p1.2.m2.1b"><ci id="Ch0.S6.I2.i1.p1.2.m2.1.1.cmml" xref="Ch0.S6.I2.i1.p1.2.m2.1.1">𝑁</ci></annotation-xml><annotation encoding="application/x-tex" id="Ch0.S6.I2.i1.p1.2.m2.1c">N</annotation><annotation encoding="application/x-llamapun" id="Ch0.S6.I2.i1.p1.2.m2.1d">italic_N</annotation></semantics></math><span class="ltx_text" id="Ch0.S6.I2.i1.p1.2.3" style="font-size:70%;"> = 30).</span></p>
</div>
</li>
</ul>
</div>
</div>
</figure>
</section>
<section class="ltx_subsection" id="Ch0.S6.SS2">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">6.2 </span>Visual Turing Test</h3>
<div class="ltx_para" id="Ch0.S6.SS2.p1">
<p class="ltx_p" id="Ch0.S6.SS2.p1.1">The Turing Test, introduced by Alan Turing in “Computing Machinery and Intelligence” <cite class="ltx_cite ltx_citemacro_cite"><a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#bib.bib98" title="">turing2009computing </a></cite>, assesses whether a machine can exhibit intelligent behavior indistinguishable from that of a human. We apply the Visual Turing Test to evaluate if synthetic tumors resemble real tumors. For this, we compared CT volumes containing real and synthetic tumors across various organs. Professionals, blinded to the origins of the samples, classified each volume as real or synthetic based on 3D views of continuous slice sequences (see Fig. <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.F9" title="Figure 9 ‣ 6.1 Case Study: Fake Tumors, Real Results ‣ 6 Tumor Synthesis Benchmark ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">9</span></a>).</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S6.SS2.p2">
<p class="ltx_p" id="Ch0.S6.SS2.p2.1"><span class="ltx_text ltx_font_bold" id="Ch0.S6.SS2.p2.1.1">Modeling-based approach (Pixel2Cancer).</span> The outcome metrics, as presented in Table <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.T3" title="Table 3 ‣ 6.1 Case Study: Fake Tumors, Real Results ‣ 6 Tumor Synthesis Benchmark ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">3</span></a>, unveil the performance evaluations by different radiologists. For the junior radiologist R1 (7 years of experience), metrics such as accuracy, sensitivity, and specificity all register below 40%. Notably, a specificity of 35.5% indicates that 64.5% of synthetic tumors are inaccurately identified as real. The intermediate radiologist R2 (9 years of experience) exhibits comparable metrics around 40%, with 59.2% of synthetic tumors causing confusion. Even the senior radiologist R3 (14 years of experience) misclassifies 44.1% of synthetic tumors as real, underscoring the formidable challenge posed even to seasoned professionals. These results emphasize the efficacy of our modeling-based approach (Pixel2Cancer) in achieving a realistic simulation of tumor development.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S6.SS2.p3">
<p class="ltx_p" id="Ch0.S6.SS2.p3.1"><span class="ltx_text ltx_font_bold" id="Ch0.S6.SS2.p3.1.1">Learning-based approach (DiffTumor).</span> Two professionals were involved in this test, with 4 and 11 years of experience, respectively. 60 CT volumes were arranged in random order and scrutinized independently by two professionals. The test outcomes are detailed in Table <a class="ltx_ref" href="https://arxiv.org/html/2409.06035v1#Ch0.T3" title="Table 3 ‣ 6.1 Case Study: Fake Tumors, Real Results ‣ 6 Tumor Synthesis Benchmark ‣ Analyzing Tumors by Synthesis"><span class="ltx_text ltx_ref_tag">3</span></a>.
Radiologists R1’s near-zero specificity scores suggested that synthetic data closely resembled real tumors, resulting in the misclassification of most synthetic tumors as real. Consequently, R1’s accuracy scores hovered around 50%. Conversely, R2, with more experience, exhibited higher specificity scores compared to R1, approaching 50%. This implies that nearly 50% of synthetic samples were correctly identified as synthetic, indicating a better discernment between real and synthetic tumors by R2. These findings affirm the effectiveness of DiffTumor in generating visually realistic tumors.</p>
</div>
</section>
</section>
<section class="ltx_section" id="Ch0.S7">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">7 </span>Conclusion</h2>
<div class="ltx_para" id="Ch0.S7.p1">
<p class="ltx_p" id="Ch0.S7.p1.1">In this chapter, we delved into the concept of synthetic data and its application in medical fields, with a particular focus on cancer research. We defined synthetic data and discussed its critical role in cancer research, such as improving data diversity, protecting patient privacy, and enabling robust research in tumor detection, diagnosis, and treatment. We explored the challenges and opportunities presented in cancer research. Despite these challenges, synthetic data offers significant opportunities, such as enhancing the training of machine learning models, supporting large-scale studies without privacy concerns, and fostering innovation in personalized medicine. Additionally, we highlighted promising approaches and future directions for the use of synthetic data in cancer research, including modeling-based methods and learning-based methods. These methods are opening new avenues for more accurate and comprehensive cancer research, enabling researchers to simulate various scenarios and treatments, and ultimately contributing to better patient outcomes.</p>
</div>
<div class="ltx_para ltx_noindent" id="Ch0.S7.p2">
<p class="ltx_p" id="Ch0.S7.p2.1"><span class="ltx_text ltx_font_bold" id="Ch0.S7.p2.1.1">Acknowledgement.</span> This work was supported by the Lustgarten Foundation for Pancreatic Cancer Research and the Patrick J. McGovern Foundation Award.</p>
</div>
</section>
<section class="ltx_bibliography" id="bib">
<h2 class="ltx_title ltx_title_bibliography">References</h2>
<ul class="ltx_biblist">
<li class="ltx_bibitem" id="bib.bib1">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(1)</span>
<span class="ltx_bibblock">
Abi Nader, C., Vetil, R., Wood, L.K., Rohe, M.M., Bône, A., Karteszi, H., Vullierme, M.P.: Automatic detection of pancreatic lesions and main pancreatic duct dilatation on portal venous ct scans using deep learning.

</span>
<span class="ltx_bibblock">Investigative Radiology (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib2">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(2)</span>
<span class="ltx_bibblock">
Antonelli, M., Reinke, A., Bakas, S., Farahani, K., Kopp-Schneider, A., Landman, B.A., Litjens, G., Menze, B., Ronneberger, O., Summers, R.M., et al.: The medical segmentation decathlon.

</span>
<span class="ltx_bibblock">Nature communications <span class="ltx_text ltx_font_bold" id="bib.bib2.1.1">13</span>(1), 1–13 (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib3">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(3)</span>
<span class="ltx_bibblock">
Armato III, S.G., McLennan, G., Bidaut, L., McNitt-Gray, M.F., Meyer, C.R., Reeves, A.P., Zhao, B., Aberle, D.R., Henschke, C.I., Hoffman, E.A., et al.: The lung image database consortium (lidc) and image database resource initiative (idri): a completed reference database of lung nodules on ct scans.

</span>
<span class="ltx_bibblock">Medical physics <span class="ltx_text ltx_font_bold" id="bib.bib3.1.1">38</span>(2), 915–931 (2011)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib4">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(4)</span>
<span class="ltx_bibblock">
Aversa, M., Nobis, G., Hägele, M., Standvoss, K., Chirica, M., Murray-Smith, R., Alaa, A.M., Ruff, L., Ivanova, D., Samek, W., et al.: Diffinfinite: Large mask-image synthesis via parallel random patch diffusion in histopathology.

</span>
<span class="ltx_bibblock">Advances in Neural Information Processing Systems <span class="ltx_text ltx_font_bold" id="bib.bib4.1.1">36</span> (2024)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib5">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(5)</span>
<span class="ltx_bibblock">
Ayuso, C., Rimola, J., Vilana, R., Burrel, M., Darnell, A., García-Criado, Á., Bianchi, L., Belmonte, E., Caparroz, C., Barrufet, M., et al.: Diagnosis and staging of hepatocellular carcinoma (hcc): current guidelines.

</span>
<span class="ltx_bibblock">European journal of radiology <span class="ltx_text ltx_font_bold" id="bib.bib5.1.1">101</span>, 72–81 (2018)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib6">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(6)</span>
<span class="ltx_bibblock">
Basaran, B.D., Zhang, W., Qiao, M., Kainz, B., Matthews, P.M., Bai, W.: Lesionmix: A lesion-level data augmentation method for medical image segmentation.

</span>
<span class="ltx_bibblock">In: International Conference on Medical Image Computing and Computer-Assisted Intervention, pp. 73–83. Springer (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib7">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(7)</span>
<span class="ltx_bibblock">
Baur, C., Denner, S., Wiestler, B., Navab, N., Albarqouni, S.: Autoencoders for unsupervised anomaly segmentation in brain mr images: a comparative study.

</span>
<span class="ltx_bibblock">Medical Image Analysis <span class="ltx_text ltx_font_bold" id="bib.bib7.1.1">69</span>, 101952 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib8">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(8)</span>
<span class="ltx_bibblock">
Bilic, P., Christ, P., Li, H.B., Vorontsov, E., Ben-Cohen, A., Kaissis, G., Szeskin, A., Jacobs, C., Mamani, G.E.H., Chartrand, G., et al.: The liver tumor segmentation benchmark (lits).

</span>
<span class="ltx_bibblock">Medical Image Analysis <span class="ltx_text ltx_font_bold" id="bib.bib8.1.1">84</span>, 102680 (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib9">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(9)</span>
<span class="ltx_bibblock">
Chen, Q., Chen, X., Song, H., Xiong, Z., Yuille, A., Wei, C., Zhou, Z.: Towards generalizable tumor synthesis.

</span>
<span class="ltx_bibblock">In: IEEE/CVF Conference on Computer Vision and Pattern Recognition (2024).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/DiffTumor" title="">https://github.com/MrGiovanni/DiffTumor</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib10">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(10)</span>
<span class="ltx_bibblock">
Chiruvella, V., Guddati, A.K., et al.: Ethical issues in patient data ownership.

</span>
<span class="ltx_bibblock">Interactive journal of medical research <span class="ltx_text ltx_font_bold" id="bib.bib10.1.1">10</span>(2), e22269 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib11">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(11)</span>
<span class="ltx_bibblock">
Chlap, P., Min, H., Vandenberg, N., Dowling, J., Holloway, L., Haworth, A.: A review of medical image data augmentation techniques for deep learning applications.

</span>
<span class="ltx_bibblock">Journal of Medical Imaging and Radiation Oncology <span class="ltx_text ltx_font_bold" id="bib.bib11.1.1">65</span>(5), 545–563 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib12">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(12)</span>
<span class="ltx_bibblock">
Choi, J.Y., Lee, J.M., Sirlin, C.B.: Ct and mr imaging diagnosis and staging of hepatocellular carcinoma: part i. development, growth, and spread: key pathologic and imaging aspects.

</span>
<span class="ltx_bibblock">Radiology <span class="ltx_text ltx_font_bold" id="bib.bib12.1.1">272</span>(3), 635–654 (2014)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib13">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(13)</span>
<span class="ltx_bibblock">
Chou, Y.C., Li, B., Fan, D.P., Yuille, A., Zhou, Z.: Acquiring weak annotations for tumor localization in temporal and volumetric data.

</span>
<span class="ltx_bibblock">Machine Intelligence Research pp. 1–13 (2024).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/johnson111788/Drag-Drop" title="">https://github.com/johnson111788/Drag-Drop</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib14">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(14)</span>
<span class="ltx_bibblock">
Chou, Y.C., Zhou, Z., Yuille, A.: Embracing massive medical data.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2407.04687 (2024).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/OnlineLearning" title="">https://github.com/MrGiovanni/OnlineLearning</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib15">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(15)</span>
<span class="ltx_bibblock">
Chu, L.C., Goggins, M.G., Fishman, E.K.: Diagnosis and detection of pancreatic cancer.

</span>
<span class="ltx_bibblock">The Cancer Journal <span class="ltx_text ltx_font_bold" id="bib.bib15.1.1">23</span>(6), 333–342 (2017)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib16">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(16)</span>
<span class="ltx_bibblock">
Chu, L.C., Park, S., Kawamoto, S., Fouladi, D.F., Shayesteh, S., Zinreich, E.S., Graves, J.S., Horton, K.M., Hruban, R.H., Yuille, A.L., et al.: Utility of ct radiomics features in differentiation of pancreatic ductal adenocarcinoma from normal pancreatic tissue.

</span>
<span class="ltx_bibblock">American Journal of Roentgenology <span class="ltx_text ltx_font_bold" id="bib.bib16.1.1">213</span>(2), 349–357 (2019)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib17">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(17)</span>
<span class="ltx_bibblock">
Codella, N.C., Gutman, D., Celebi, M.E., Helba, B., Marchetti, M.A., Dusza, S.W., Kalloo, A., Liopyris, K., Mishra, N., Kittler, H., et al.: Skin lesion analysis toward melanoma detection: A challenge at the 2017 international symposium on biomedical imaging (isbi), hosted by the international skin imaging collaboration (isic).

</span>
<span class="ltx_bibblock">In: 2018 IEEE 15th international symposium on biomedical imaging (ISBI 2018), pp. 168–172. IEEE (2018)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib18">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(18)</span>
<span class="ltx_bibblock">
Davis, G.B., Blanchard, D.K., Hatch III, G.F., Wertheimer-Hatch, L., Hatch, K.F., Foster Jr, R.S., Skandalakis, J.E.: Tumors of the stomach.

</span>
<span class="ltx_bibblock">World journal of surgery <span class="ltx_text ltx_font_bold" id="bib.bib18.1.1">24</span>(4), 412–420 (2000)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib19">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(19)</span>
<span class="ltx_bibblock">
Du, S., Wang, X., Lu, Y., Zhou, Y., Zhang, S., Yuille, A., Li, K., Zhou, Z.: Boosting dermatoscopic lesion segmentation via diffusion models with visual and textual prompts.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2310.02906 (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib20">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(20)</span>
<span class="ltx_bibblock">
Dunnick, N.R.: Renal cell carcinoma: staging and surveillance.

</span>
<span class="ltx_bibblock">Abdominal Radiology <span class="ltx_text ltx_font_bold" id="bib.bib20.1.1">41</span>, 1079–1085 (2016)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib21">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(21)</span>
<span class="ltx_bibblock">
Edge, S.B., Compton, C.C.: The american joint committee on cancer: the 7th edition of the ajcc cancer staging manual and the future of tnm.

</span>
<span class="ltx_bibblock">Annals of surgical oncology <span class="ltx_text ltx_font_bold" id="bib.bib21.1.1">17</span>(6), 1471–1474 (2010)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib22">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(22)</span>
<span class="ltx_bibblock">
Elbanna, K.Y., Jang, H.J., Kim, T.K.: Imaging diagnosis and staging of pancreatic ductal adenocarcinoma: a comprehensive review.

</span>
<span class="ltx_bibblock">Insights into imaging <span class="ltx_text ltx_font_bold" id="bib.bib22.1.1">11</span>(1), 1–13 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib23">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(23)</span>
<span class="ltx_bibblock">
Engelhardt, S., Sauerzapf, S., Preim, B., Karck, M., Wolf, I., De Simone, R.: Flexible and comprehensive patient-specific mitral valve silicone models with chordae tendineae made from 3d-printable molds.

</span>
<span class="ltx_bibblock">International journal of computer assisted radiology and surgery <span class="ltx_text ltx_font_bold" id="bib.bib23.1.1">14</span>, 1177–1186 (2019)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib24">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(24)</span>
<span class="ltx_bibblock">
Feng, R., Zhou, Z., Gotway, M.B., Liang, J.: Parts2whole: Self-supervised contrastive learning via reconstruction.

</span>
<span class="ltx_bibblock">In: Domain Adaptation and Representation Transfer, and Distributed and Collaborative Learning, pp. 85–95. Springer (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib25">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(25)</span>
<span class="ltx_bibblock">
Fowler, K.J., Burgoyne, A., Fraum, T.J., Hosseini, M., Ichikawa, S., Kim, S., Kitao, A., Lee, J.M., Paradis, V., Taouli, B., et al.: Pathologic, molecular, and prognostic radiologic features of hepatocellular carcinoma.

</span>
<span class="ltx_bibblock">Radiographics <span class="ltx_text ltx_font_bold" id="bib.bib25.1.1">41</span>(6), 1611–1631 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib26">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(26)</span>
<span class="ltx_bibblock">
Gao, C., Killeen, B.D., Hu, Y., Grupp, R.B., Taylor, R.H., Armand, M., Unberath, M.: Synthetic data accelerates the development of generalizable learning-based algorithms for x-ray image analysis.

</span>
<span class="ltx_bibblock">Nature Machine Intelligence <span class="ltx_text ltx_font_bold" id="bib.bib26.1.1">5</span>(3), 294–308 (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib27">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(27)</span>
<span class="ltx_bibblock">
Golias, C., Charalabopoulos, A., Charalabopoulos, K.: Cell proliferation and cell cycle control: a mini review.

</span>
<span class="ltx_bibblock">International journal of clinical practice <span class="ltx_text ltx_font_bold" id="bib.bib27.1.1">58</span>(12), 1134–1141 (2004)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib28">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(28)</span>
<span class="ltx_bibblock">
Goodfellow, I., Pouget-Abadie, J., Mirza, M., Xu, B., Warde-Farley, D., Ozair, S., Courville, A., Bengio, Y.: Generative adversarial nets.

</span>
<span class="ltx_bibblock">Advances in neural information processing systems <span class="ltx_text ltx_font_bold" id="bib.bib28.1.1">27</span> (2014)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib29">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(29)</span>
<span class="ltx_bibblock">
Goodfellow, I., Pouget-Abadie, J., Mirza, M., Xu, B., Warde-Farley, D., Ozair, S., Courville, A., Bengio, Y.: Generative adversarial networks.

</span>
<span class="ltx_bibblock">Communications of the ACM <span class="ltx_text ltx_font_bold" id="bib.bib29.1.1">63</span>(11), 139–144 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib30">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(30)</span>
<span class="ltx_bibblock">
Griswold, D., Corbett, T.H.: A colon tumor model for anticancer agent evaluation.

</span>
<span class="ltx_bibblock">Cancer <span class="ltx_text ltx_font_bold" id="bib.bib30.1.1">36</span>(S6), 2441–2444 (1975)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib31">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(31)</span>
<span class="ltx_bibblock">
Haghighi, F., Hosseinzadeh Taher, M.R., Zhou, Z., Gotway, M.B., Liang, J.: Learning semantics-enriched representation via self-discovery, self-classification, and self-restoration.

</span>
<span class="ltx_bibblock">In: International Conference on Medical Image Computing and Computer-Assisted Intervention, pp. 137–147. Springer (2020).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/fhaghighi/SemanticGenesis" title="">https://github.com/fhaghighi/SemanticGenesis</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib32">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(32)</span>
<span class="ltx_bibblock">
Haghighi, F., Taher, M.R.H., Zhou, Z., Gotway, M.B., Liang, J.: Transferable visual words: Exploiting the semantics of anatomical patterns for self-supervised learning.

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging (2021).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/fhaghighi/SemanticGenesis" title="">https://github.com/fhaghighi/SemanticGenesis</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib33">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(33)</span>
<span class="ltx_bibblock">
Hamamci, I.E., Er, S., Simsar, E., Tezcan, A., Simsek, A.G., Almas, F., Esirgun, S.N., Reynaud, H., Pati, S., Bluethgen, C., et al.: Generatect: text-guided 3d chest ct generation.

</span>
<span class="ltx_bibblock">In: Proceedings of the European Conference on Computer Vision (ECCV) (2024)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib34">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(34)</span>
<span class="ltx_bibblock">
Han, C., Rundo, L., Araki, R., Furukawa, Y., Mauri, G., Nakayama, H., Hayashi, H.: Infinite brain mr images: Pggan-based data augmentation for tumor detection.

</span>
<span class="ltx_bibblock">In: Neural approaches to dynamics of signal exchanges, pp. 291–303. Springer (2019)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib35">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(35)</span>
<span class="ltx_bibblock">
Hanahan, D., Weinberg, R.A.: The hallmarks of cancer.

</span>
<span class="ltx_bibblock">cell <span class="ltx_text ltx_font_bold" id="bib.bib35.1.1">100</span>(1), 57–70 (2000)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib36">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(36)</span>
<span class="ltx_bibblock">
Harpold, H.L., Alvord Jr, E.C., Swanson, K.R.: The evolution of mathematical modeling of glioma proliferation and invasion.

</span>
<span class="ltx_bibblock">Journal of Neuropathology &amp; Experimental Neurology <span class="ltx_text ltx_font_bold" id="bib.bib36.1.1">66</span>(1), 1–9 (2007)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib37">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(37)</span>
<span class="ltx_bibblock">
Heath, M., Bowyer, K., Kopans, D., Kegelmeyer Jr, P., Moore, R., Chang, K., Munishkumaran, S.: Current status of the digital database for screening mammography.

</span>
<span class="ltx_bibblock">In: Digital Mammography: Nijmegen, 1998, pp. 457–460. Springer (1998)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib38">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(38)</span>
<span class="ltx_bibblock">
Heller, N., Isensee, F., Maier-Hein, K.H., Hou, X., Xie, C., Li, F., Nan, Y., Mu, G., Lin, Z., Han, M., et al.: The state of the art in kidney and kidney tumor segmentation in contrast-enhanced ct imaging: Results of the kits19 challenge.

</span>
<span class="ltx_bibblock">Medical image analysis <span class="ltx_text ltx_font_bold" id="bib.bib38.1.1">67</span>, 101821 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib39">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(39)</span>
<span class="ltx_bibblock">
Hiraoka, N., Ino, Y., Sekine, S., Tsuda, H., Shimada, K., Kosuge, T., Zavada, J., Yoshida, M., Yamada, K., Koyama, T., et al.: Tumour necrosis is a postoperative prognostic marker for pancreatic cancer patients with a high interobserver reproducibility in histological evaluation.

</span>
<span class="ltx_bibblock">British journal of cancer <span class="ltx_text ltx_font_bold" id="bib.bib39.1.1">103</span>(7), 1057–1065 (2010)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib40">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(40)</span>
<span class="ltx_bibblock">
Ho, J., Jain, A., Abbeel, P.: Denoising diffusion probabilistic models.

</span>
<span class="ltx_bibblock">Advances in Neural Information Processing Systems <span class="ltx_text ltx_font_bold" id="bib.bib40.1.1">33</span>, 6840–6851 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib41">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(41)</span>
<span class="ltx_bibblock">
Hou, L., Agarwal, A., Samaras, D., Kurc, T.M., Gupta, R.R., Saltz, J.H.: Robust histopathology image analysis: To label or to synthesize?

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE/CVF conference on computer vision and pattern recognition, pp. 8533–8542 (2019)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib42">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(42)</span>
<span class="ltx_bibblock">
Hu, Q., Chen, Y., Xiao, J., Sun, S., Chen, J., Yuille, A.L., Zhou, Z.: Label-free liver tumor segmentation.

</span>
<span class="ltx_bibblock">In: IEEE/CVF Conference on Computer Vision and Pattern Recognition, pp. 7422–7432 (2023).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/SyntheticTumors" title="">https://github.com/MrGiovanni/SyntheticTumors</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib43">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(43)</span>
<span class="ltx_bibblock">
Hu, Q., Xiao, J., Chen, Y., Sun, S., Chen, J.N., Yuille, A., Zhou, Z.: Synthetic tumors make ai segment tumors better.

</span>
<span class="ltx_bibblock">NeurIPS Workshop on Medical Imaging meets NeurIPS (2022).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/SyntheticTumors" title="">https://github.com/MrGiovanni/SyntheticTumors</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib44">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(44)</span>
<span class="ltx_bibblock">
Hu, Q., Yuille, A., Zhou, Z.: Synthetic data as validation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2310.16052 (2023).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/SyntheticValidation" title="">https://github.com/MrGiovanni/SyntheticValidation</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib45">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(45)</span>
<span class="ltx_bibblock">
Iyer, R., Dubrow, R.: Imaging of esophageal cancer.

</span>
<span class="ltx_bibblock">Cancer Imaging <span class="ltx_text ltx_font_bold" id="bib.bib45.1.1">4</span>(2), 125 (2004)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib46">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(46)</span>
<span class="ltx_bibblock">
Jaipuria, N., Zhang, X., Bhasin, R., Arafa, M., Chakravarty, P., Shrivastava, S., Manglani, S., Murali, V.N.: Deflating dataset bias using synthetic data augmentation.

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition Workshops, pp. 772–773 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib47">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(47)</span>
<span class="ltx_bibblock">
Jiang, Y., Chen, H., Loew, M., Ko, H.: Covid-19 ct image synthesis with a conditional generative adversarial network.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics <span class="ltx_text ltx_font_bold" id="bib.bib47.1.1">25</span>(2), 441–452 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib48">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(48)</span>
<span class="ltx_bibblock">
Jin, D., Xu, Z., Tang, Y., Harrison, A.P., Mollura, D.J.: Ct-realistic lung nodule simulation from 3d conditional generative adversarial networks for robust lung segmentation.

</span>
<span class="ltx_bibblock">In: Medical Image Computing and Computer Assisted Intervention, pp. 732–740. Springer (2018)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib49">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(49)</span>
<span class="ltx_bibblock">
Jin, Q., Cui, H., Sun, C., Meng, Z., Su, R.: Free-form tumor synthesis in computed tomography images via richer generative adversarial network.

</span>
<span class="ltx_bibblock">Knowledge-Based Systems <span class="ltx_text ltx_font_bold" id="bib.bib49.1.1">218</span>, 106753 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib50">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(50)</span>
<span class="ltx_bibblock">
Kang, M., Li, B., Zhu, Z., Lu, Y., Fishman, E.K., Yuille, A., Zhou, Z.: Label-assemble: Leveraging multiple datasets with partial labels.

</span>
<span class="ltx_bibblock">In: IEEE International Symposium on Biomedical Imaging, pp. 1–5. IEEE (2023).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/LabelAssemble" title="">https://github.com/MrGiovanni/LabelAssemble</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib51">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(51)</span>
<span class="ltx_bibblock">
Kang, M.J., Jang, J.Y., Kim, S.J., Lee, K.B., Ryu, J.K., Kim, Y.T., Yoon, Y.B., Kim, S.W.: Cyst growth rate predicts malignancy in patients with branch duct intraductal papillary mucinous neoplasms.

</span>
<span class="ltx_bibblock">Clinical Gastroenterology and Hepatology <span class="ltx_text ltx_font_bold" id="bib.bib51.1.1">9</span>(1), 87–93 (2011)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib52">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(52)</span>
<span class="ltx_bibblock">
Khader, F., Müller-Franzes, G., Tayebi Arasteh, S., Han, T., Haarburger, C., Schulze-Hagen, M., Schad, P., Engelhardt, S., Baeßler, B., Foersch, S., et al.: Denoising diffusion probabilistic models for 3d medical image generation.

</span>
<span class="ltx_bibblock">Scientific Reports <span class="ltx_text ltx_font_bold" id="bib.bib52.1.1">13</span>(1), 7303 (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib53">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(53)</span>
<span class="ltx_bibblock">
Kingma, D.P., Welling, M.: Auto-encoding variational bayes.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:1312.6114 (2013)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib54">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(54)</span>
<span class="ltx_bibblock">
Kuijf, H.J., Biesbroek, J.M., De Bresser, J., Heinen, R., Andermatt, S., Bento, M., Berseth, M., Belyaev, M., Cardoso, M.J., Casamitjana, A., et al.: Standardized assessment of automatic segmentation of white matter hyperintensities and results of the wmh segmentation challenge.

</span>
<span class="ltx_bibblock">IEEE transactions on medical imaging <span class="ltx_text ltx_font_bold" id="bib.bib54.1.1">38</span>(11), 2556–2568 (2019)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib55">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(55)</span>
<span class="ltx_bibblock">
Kumar, N., Verma, R., Sharma, S., Bhargava, S., Vahadane, A., Sethi, A.: A dataset and a technique for generalized nuclear segmentation for computational pathology.

</span>
<span class="ltx_bibblock">IEEE transactions on medical imaging <span class="ltx_text ltx_font_bold" id="bib.bib55.1.1">36</span>(7), 1550–1560 (2017)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib56">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(56)</span>
<span class="ltx_bibblock">
Kumar, V., Abbas, A., Aster, J.C.: Robbins basic pathology.

</span>
<span class="ltx_bibblock">Elsevier Health Sciences (2017)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib57">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(57)</span>
<span class="ltx_bibblock">
Laeseke, P.F., Chen, R., Jeffrey, R.B., Brentnall, T.A., Willmann, J.K.: Combining in vitro diagnostics with in vivo imaging for earlier detection of pancreatic ductal adenocarcinoma: challenges and solutions.

</span>
<span class="ltx_bibblock">Radiology <span class="ltx_text ltx_font_bold" id="bib.bib57.1.1">277</span>(3), 644–661 (2015)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib58">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(58)</span>
<span class="ltx_bibblock">
Lahouel, K., Younes, L., Danilova, L., Giardiello, F.M., Hruban, R.H., Groopman, J., Kinzler, K.W., Vogelstein, B., Geman, D., Tomasetti, C.: Revisiting the tumorigenesis timeline with a data-driven generative model.

</span>
<span class="ltx_bibblock">Proceedings of the National Academy of Sciences <span class="ltx_text ltx_font_bold" id="bib.bib58.1.1">117</span>(2), 857–864 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib59">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(59)</span>
<span class="ltx_bibblock">
Lai, Y., Chen, X., Wang, A., Yuille, A., Zhou, Z.: From pixel to cancer: Cellular automata in computed tomography.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2403.06459 (2024).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/Pixel2Cancer" title="">https://github.com/MrGiovanni/Pixel2Cancer</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib60">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(60)</span>
<span class="ltx_bibblock">
Lee, K., O’Malley, M., Haider, M., Hanbidge, A.: Triple-phase mdct of hepatocellular carcinoma.

</span>
<span class="ltx_bibblock">American Journal of Roentgenology <span class="ltx_text ltx_font_bold" id="bib.bib60.1.1">182</span>(3), 643–649 (2004)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib61">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(61)</span>
<span class="ltx_bibblock">
Leveridge, M.J., Bostrom, P.J., Koulouris, G., Finelli, A., Lawrentschuk, N.: Imaging renal cell carcinoma with ultrasonography, ct and mri.

</span>
<span class="ltx_bibblock">Nature Reviews Urology <span class="ltx_text ltx_font_bold" id="bib.bib61.1.1">7</span>(6), 311–325 (2010)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib62">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(62)</span>
<span class="ltx_bibblock">
Lewin, K.J., Appelman, H.D., et al.: Tumors of the esophagus and stomach.

</span>
<span class="ltx_bibblock">American Registry of Pathology (1996)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib63">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(63)</span>
<span class="ltx_bibblock">
Li, B., Chou, Y.C., Sun, S., Qiao, H., Yuille, A., Zhou, Z.: Early detection and localization of pancreatic cancer by label-free tumor synthesis.

</span>
<span class="ltx_bibblock">MICCAI Workshop on Big Task Small Data, 1001-AI (2023).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/SyntheticTumors" title="">https://github.com/MrGiovanni/SyntheticTumors</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib64">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(64)</span>
<span class="ltx_bibblock">
Li, C., Liu, H., Liu, Y., Feng, B.Y., Li, W., Liu, X., Chen, Z., Shao, J., Yuan, Y.: Endora: Video generation models as endoscopy simulators.

</span>
<span class="ltx_bibblock">In: International Conference on Medical Image Computing and Computer-Assisted Intervention (2024)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib65">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(65)</span>
<span class="ltx_bibblock">
Li, W., Qu, C., Chen, X., Bassi, P.R., Shi, Y., Lai, Y., Yu, Q., Xue, H., Chen, Y., Lin, X., et al.: Abdomenatlas: A large-scale, detailed-annotated, &amp; multi-center dataset for efficient transfer learning and open algorithmic benchmarking.

</span>
<span class="ltx_bibblock">Medical Image Analysis p. 103285 (2024).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/AbdomenAtlas" title="">https://github.com/MrGiovanni/AbdomenAtlas</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib66">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(66)</span>
<span class="ltx_bibblock">
Li, W., Yuille, A., Zhou, Z.: How well do supervised models transfer to 3d image segmentation?

</span>
<span class="ltx_bibblock">In: International Conference on Learning Representations (2024).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/SuPreM" title="">https://github.com/MrGiovanni/SuPreM</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib67">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(67)</span>
<span class="ltx_bibblock">
Liu, J., Zhang, Y., Chen, J.N., Xiao, J., Lu, Y., A Landman, B., Yuan, Y., Yuille, A., Tang, Y., Zhou, Z.: Clip-driven universal model for organ segmentation and tumor detection.

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE/CVF International Conference on Computer Vision, pp. 21152–21164 (2023).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/ljwztc/CLIP-Driven-Universal-Model" title="">https://github.com/ljwztc/CLIP-Driven-Universal-Model</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib68">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(68)</span>
<span class="ltx_bibblock">
Liu, J., Zhang, Y., Wang, K., Yavuz, M.C., Chen, X., Yuan, Y., Li, H., Yang, Y., Yuille, A., Tang, Y., et al.: Universal and extensible language-vision models for organ segmentation and tumor detection from abdominal computed tomography.

</span>
<span class="ltx_bibblock">Medical Image Analysis p. 103226 (2024).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/ljwztc/CLIP-Driven-Universal-Model" title="">https://github.com/ljwztc/CLIP-Driven-Universal-Model</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib69">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(69)</span>
<span class="ltx_bibblock">
Luo, Y., Zhou, L., Zhan, B., Fei, Y., Zhou, J., Wang, Y., Shen, D.: Adaptive rectification based adversarial network with spectrum constraint for high-quality pet image synthesis.

</span>
<span class="ltx_bibblock">Medical Image Analysis <span class="ltx_text ltx_font_bold" id="bib.bib69.1.1">77</span>, 102335 (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib70">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(70)</span>
<span class="ltx_bibblock">
Lyu, F., Ye, M., Ma, A.J., Yip, T.C.F., Wong, G.L.H., Yuen, P.C.: Learning from synthetic ct images via test-time training for liver tumor segmentation.

</span>
<span class="ltx_bibblock">IEEE transactions on medical imaging <span class="ltx_text ltx_font_bold" id="bib.bib70.1.1">41</span>(9), 2510–2520 (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib71">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(71)</span>
<span class="ltx_bibblock">
M. Cunha, G., Fowler, K.J., Roudenko, A., Taouli, B., Fung, A.W., Elsayes, K.M., Marks, R.M., Cruite, I., Horvat, N., Chernyak, V., et al.: How to use li-rads to report liver ct and mri observations.

</span>
<span class="ltx_bibblock">RadioGraphics <span class="ltx_text ltx_font_bold" id="bib.bib71.1.1">41</span>(5), 1352–1367 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib72">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(72)</span>
<span class="ltx_bibblock">
Menze, B.H., Jakab, A., Bauer, S., Kalpathy-Cramer, J., Farahani, K., Kirby, J., Burren, Y., Porz, N., Slotboom, J., Wiest, R., et al.: The multimodal brain tumor image segmentation benchmark (brats).

</span>
<span class="ltx_bibblock">IEEE transactions on medical imaging <span class="ltx_text ltx_font_bold" id="bib.bib72.1.1">34</span>(10), 1993–2024 (2014)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib73">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(73)</span>
<span class="ltx_bibblock">
Mukherkjee, D., Saha, P., Kaplun, D., Sinitca, A., Sarkar, R.: Brain tumor image generation using an aggregation of gan models with style transfer.

</span>
<span class="ltx_bibblock">Scientific Reports <span class="ltx_text ltx_font_bold" id="bib.bib73.1.1">12</span>(1), 9141 (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib74">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(74)</span>
<span class="ltx_bibblock">
Nerad, E., Lahaye, M.J., Maas, M., Nelemans, P., Bakers, F.C., Beets, G.L., Beets-Tan, R.G.: Diagnostic accuracy of ct for local staging of colon cancer: a systematic review and meta-analysis.

</span>
<span class="ltx_bibblock">American Journal of Roentgenology <span class="ltx_text ltx_font_bold" id="bib.bib74.1.1">207</span>(5), 984–995 (2016)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib75">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(75)</span>
<span class="ltx_bibblock">
Orbes-Arteaga, M., Varsavsky, T., Sudre, C.H., Eaton-Rosen, Z., Haddow, L.J., Sørensen, L., Nielsen, M., Pai, A., Ourselin, S., Modat, M., et al.: Multi-domain adaptation in brain mri through paired consistency and adversarial learning.

</span>
<span class="ltx_bibblock">In: Domain Adaptation and Representation Transfer and Medical Image Learning with Less Labels and Imperfect Data, pp. 54–62. Springer (2019)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib76">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(76)</span>
<span class="ltx_bibblock">
Özbey, M., Dalmaz, O., Dar, S.U., Bedel, H.A., Özturk, Ş., Güngör, A., Çukur, T.: Unsupervised medical image translation with adversarial diffusion models.

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging (2023)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib77">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(77)</span>
<span class="ltx_bibblock">
Park, J.E., Eun, D., Kim, H.S., Lee, D.H., Jang, R.W., Kim, N.: Generative adversarial network for glioblastoma ensures morphologic variations and improves diagnostic model for isocitrate dehydrogenase mutant type.

</span>
<span class="ltx_bibblock">Scientific reports <span class="ltx_text ltx_font_bold" id="bib.bib77.1.1">11</span>(1), 9912 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib78">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(78)</span>
<span class="ltx_bibblock">
Park, S., Chu, L., Fishman, E., Yuille, A., Vogelstein, B., Kinzler, K., Horton, K., Hruban, R., Zinreich, E., Fouladi, D.F., et al.: Annotated normal ct data of the abdomen for deep learning: Challenges and strategies for implementation.

</span>
<span class="ltx_bibblock">Diagnostic and interventional imaging <span class="ltx_text ltx_font_bold" id="bib.bib78.1.1">101</span>(1), 35–44 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib79">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(79)</span>
<span class="ltx_bibblock">
Qu, C., Zhang, T., Qiao, H., Liu, J., Tang, Y., Yuille, A., Zhou, Z.: Abdomenatlas-8k: Annotating 8,000 abdominal ct volumes for multi-organ segmentation in three weeks.

</span>
<span class="ltx_bibblock">Conference on Neural Information Processing Systems (2023).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/AbdomenAtlas" title="">https://github.com/MrGiovanni/AbdomenAtlas</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib80">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(80)</span>
<span class="ltx_bibblock">
Qu, C., Zhang, T., Qiao, H., Tang, Y., Yuille, A.L., Zhou, Z., et al.: Abdomenatlas-8k: Annotating 8,000 ct volumes for multi-organ segmentation in three weeks.

</span>
<span class="ltx_bibblock">Advances in Neural Information Processing Systems <span class="ltx_text ltx_font_bold" id="bib.bib80.1.1">36</span> (2024)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib81">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(81)</span>
<span class="ltx_bibblock">
Ramesh, A., Dhariwal, P., Nichol, A., Chu, C., Chen, M.: Hierarchical text-conditional image generation with clip latents.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2204.06125 <span class="ltx_text ltx_font_bold" id="bib.bib81.1.1">1</span>(2), 3 (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib82">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(82)</span>
<span class="ltx_bibblock">
Reynolds, A.R., Furlan, A., Fetzer, D.T., Sasatomi, E., Borhani, A.A., Heller, M.T., Tublin, M.E.: Infiltrative hepatocellular carcinoma: what radiologists need to know.

</span>
<span class="ltx_bibblock">Radiographics <span class="ltx_text ltx_font_bold" id="bib.bib82.1.1">35</span>(2), 371–386 (2015)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib83">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(83)</span>
<span class="ltx_bibblock">
Roth, K., Pemula, L., Zepeda, J., Schölkopf, B., Brox, T., Gehler, P.: Towards total recall in industrial anomaly detection.

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition, pp. 14318–14328 (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib84">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(84)</span>
<span class="ltx_bibblock">
Saar, B., Kellner-Weldon, F.: Radiological diagnosis of hepatocellular carcinoma.

</span>
<span class="ltx_bibblock">Liver International <span class="ltx_text ltx_font_bold" id="bib.bib84.1.1">28</span>(2), 189–199 (2008)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib85">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(85)</span>
<span class="ltx_bibblock">
Sahani, D.V., Samir, A.E.: Abdominal Imaging E-Book: Expert Radiology Series.

</span>
<span class="ltx_bibblock">Elsevier Health Sciences (2016)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib86">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(86)</span>
<span class="ltx_bibblock">
Schlegl, T., Seeböck, P., Waldstein, S.M., Langs, G., Schmidt-Erfurth, U.: f-anogan: Fast unsupervised anomaly detection with generative adversarial networks.

</span>
<span class="ltx_bibblock">Medical Image Analysis (2019)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib87">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(87)</span>
<span class="ltx_bibblock">
Setio, A.A.A., Traverso, A., De Bel, T., Berens, M.S., van den Bogaard, C., Cerello, P., Chen, H., Dou, Q., Fantacci, M.E., Geurts, B., et al.: Validation, comparison, and combination of algorithms for automatic detection of pulmonary nodules in computed tomography images: the luna16 challenge.

</span>
<span class="ltx_bibblock">Medical image analysis <span class="ltx_text ltx_font_bold" id="bib.bib87.1.1">42</span>, 1–13 (2017)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib88">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(88)</span>
<span class="ltx_bibblock">
Sharan, L., Romano, G., Koehler, S., Kelm, H., Karck, M., De Simone, R., Engelhardt, S.: Mutually improved endoscopic image synthesis and landmark detection in unpaired image-to-image translation.

</span>
<span class="ltx_bibblock">IEEE Journal of Biomedical and Health Informatics <span class="ltx_text ltx_font_bold" id="bib.bib88.1.1">26</span>(1), 127–138 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib89">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(89)</span>
<span class="ltx_bibblock">
Silverman, P.M.: Oncologic imaging: a multidisciplinary approach.

</span>
<span class="ltx_bibblock">Elsevier Health Sciences (2012)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib90">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(90)</span>
<span class="ltx_bibblock">
Skarin, A.T.: Atlas of Diagnostic Oncology E-Book.

</span>
<span class="ltx_bibblock">Elsevier Health Sciences (2015)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib91">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(91)</span>
<span class="ltx_bibblock">
Smaldone, M.C., Kutikov, A., Egleston, B.L., Canter, D.J., Viterbo, R., Chen, D.Y., Jewett, M.A., Greenberg, R.E., Uzzo, R.G.: Small renal masses progressing to metastases under active surveillance: a systematic review and pooled analysis.

</span>
<span class="ltx_bibblock">Cancer <span class="ltx_text ltx_font_bold" id="bib.bib91.1.1">118</span>(4), 997–1006 (2012)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib92">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(92)</span>
<span class="ltx_bibblock">
Soga, J.: Early-stage carcinoids of the gastrointestinal tract: an analysis of 1914 reported cases.

</span>
<span class="ltx_bibblock">Cancer: Interdisciplinary International Journal of the American Cancer Society <span class="ltx_text ltx_font_bold" id="bib.bib92.1.1">103</span>(8), 1587–1595 (2005)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib93">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(93)</span>
<span class="ltx_bibblock">
Stout, A.P.: Tumors of the stomach.

</span>
<span class="ltx_bibblock">Armed Forces Institute of Pathology (1953)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib94">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(94)</span>
<span class="ltx_bibblock">
Stout, A.P., Lattes, R.: Tumors of the esophagus.

</span>
<span class="ltx_bibblock">Armed Forces Institute of Pathology (1957)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib95">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(95)</span>
<span class="ltx_bibblock">
Tanase, M., Waliszewski, P.: On complexity and homogeneity measures in predicting biological aggressiveness of prostate cancer; implication of the cellular automata model of tumor growth.

</span>
<span class="ltx_bibblock">Journal of surgical oncology <span class="ltx_text ltx_font_bold" id="bib.bib95.1.1">112</span>(8), 791–801 (2015)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib96">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(96)</span>
<span class="ltx_bibblock">
Tang, Y., Liu, J., Zhou, Z., Yu, X., Huo, Y.: Efficient 3d representation learning for medical image analysis.

</span>
<span class="ltx_bibblock">World Scientific Annual Review of Artificial Intelligence (2024)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib97">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(97)</span>
<span class="ltx_bibblock">
Teixeira, B., Singh, V., Chen, T., Ma, K., Tamersoy, B., Wu, Y., Balashova, E., Comaniciu, D.: Generating synthetic x-ray images of a person from the surface geometry.

</span>
<span class="ltx_bibblock">In: Proceedings of the IEEE conference on computer vision and pattern recognition, pp. 9059–9067 (2018)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib98">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(98)</span>
<span class="ltx_bibblock">
Turing, A.M.: Computing machinery and intelligence.

</span>
<span class="ltx_bibblock">Springer (2009)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib99">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(99)</span>
<span class="ltx_bibblock">
Van Griethuysen, J.J., Fedorov, A., Parmar, C., Hosny, A., Aucoin, N., Narayan, V., Beets-Tan, R.G., Fillion-Robin, J.C., Pieper, S., Aerts, H.J.: Computational radiomics system to decode the radiographic phenotype.

</span>
<span class="ltx_bibblock">Cancer research <span class="ltx_text ltx_font_bold" id="bib.bib99.1.1">77</span>(21), e104–e107 (2017)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib100">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(100)</span>
<span class="ltx_bibblock">
Veeling, B.S., Linmans, J., Winkens, J., Cohen, T., Welling, M.: Rotation equivariant cnns for digital pathology.

</span>
<span class="ltx_bibblock">In: Medical Image Computing and Computer Assisted Intervention, pp. 210–218. Springer (2018)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib101">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(101)</span>
<span class="ltx_bibblock">
Waheed, A., Goyal, M., Gupta, D., Khanna, A., Al-Turjman, F., Pinheiro, P.R.: Covidgan: data augmentation using auxiliary classifier gan for improved covid-19 detection.

</span>
<span class="ltx_bibblock">Ieee Access <span class="ltx_text ltx_font_bold" id="bib.bib101.1.1">8</span>, 91916–91923 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib102">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(102)</span>
<span class="ltx_bibblock">
Wang, H., Zhou, Z., Li, Y., Chen, Z., Lu, P., Wang, W., Liu, W., Yu, L.: Comparison of machine learning methods for classifying mediastinal lymph node metastasis of non-small cell lung cancer from 18f-fdg pet/ct images.

</span>
<span class="ltx_bibblock">EJNMMI research <span class="ltx_text ltx_font_bold" id="bib.bib102.1.1">7</span>(1), 1–11 (2017)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib103">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(103)</span>
<span class="ltx_bibblock">
Wang, Y., Zhou, L., Wang, L., Yu, B., Zu, C., Lalush, D.S., Lin, W., Wu, X., Zhou, J., Shen, D.: Locality adaptive multi-modality gans for high-quality pet image synthesis.

</span>
<span class="ltx_bibblock">In: Medical Image Computing and Computer Assisted Intervention, pp. 329–337. Springer (2018)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib104">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(104)</span>
<span class="ltx_bibblock">
Wang, Z.J., Westphalen, A.C., Zagoria, R.J.: CT and MRI of small renal masses.

</span>
<span class="ltx_bibblock">Br J Radiol <span class="ltx_text ltx_font_bold" id="bib.bib104.1.1">91</span>(1087), 20180131 (2018)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib105">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(105)</span>
<span class="ltx_bibblock">
Wei, Z., Chen, Y., Guan, Q., Hu, H., Zhou, Q., Li, Z., Xu, X., Frangi, A., Chen, F.: Pancreatic image augmentation based on local region texture synthesis for tumor segmentation.

</span>
<span class="ltx_bibblock">In: International Conference on Artificial Neural Networks, pp. 419–431. Springer (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib106">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(106)</span>
<span class="ltx_bibblock">
Wong, K.C., Summers, R.M., Kebebew, E., Yao, J.: Pancreatic tumor growth prediction with multiplicative growth and image-derived motion.

</span>
<span class="ltx_bibblock">In: International Conference on Information Processing in Medical Imaging, pp. 501–513. Springer (2015)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib107">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(107)</span>
<span class="ltx_bibblock">
Wu, E., Wu, K., Cox, D., Lotter, W.: Conditional infilling gans for data augmentation in mammogram classification.

</span>
<span class="ltx_bibblock">In: Image Analysis for Moving Organ, Breast, and Thoracic Images: Third International Workshop, pp. 98–106. Springer (2018)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib108">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(108)</span>
<span class="ltx_bibblock">
Xia, Y., Yu, Q., Chu, L., Kawamoto, S., Park, S., Liu, F., Chen, J., Zhu, Z., Li, B., Zhou, Z., et al.: The felix project: Deep networks to detect pancreatic neoplasms.

</span>
<span class="ltx_bibblock">medRxiv (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib109">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(109)</span>
<span class="ltx_bibblock">
Xiao, J., Yu, L., Zhou, Z., Bai, Y., Xing, L., Yuille, A., Zhou, Y.: Catenorm: Categorical normalization for robust medical image segmentation.

</span>
<span class="ltx_bibblock">In: MICCAI Workshop on Domain Adaptation and Representation Transfer, pp. 129–146. Springer (2022).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/lambert-x/CateNorm" title="">https://github.com/lambert-x/CateNorm</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib110">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(110)</span>
<span class="ltx_bibblock">
Xue, Y., Ye, J., Zhou, Q., Long, L.R., Antani, S., Xue, Z., Cornwell, C., Zaino, R., Cheng, K.C., Huang, X.: Selective synthetic augmentation with histogan for improved histopathology image classification.

</span>
<span class="ltx_bibblock">Medical image analysis <span class="ltx_text ltx_font_bold" id="bib.bib110.1.1">67</span>, 101816 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib111">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(111)</span>
<span class="ltx_bibblock">
Yan, W., Huang, L., Xia, L., Gu, S., Yan, F., Wang, Y., Tao, Q.: Mri manufacturer shift and adaptation: increasing the generalizability of deep learning segmentation for mr images acquired with different scanners.

</span>
<span class="ltx_bibblock">Radiology: Artificial Intelligence <span class="ltx_text ltx_font_bold" id="bib.bib111.1.1">2</span>(4), e190195 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib112">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(112)</span>
<span class="ltx_bibblock">
Yang, X., He, X., Zhao, J., Zhang, Y., Zhang, S., Xie, P.: Covid-ct-dataset: a ct scan dataset about covid-19.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2003.13865 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib113">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(113)</span>
<span class="ltx_bibblock">
Yao, Q., Xiao, L., Liu, P., Zhou, S.K.: Label-free segmentation of covid-19 lesions in lung ct.

</span>
<span class="ltx_bibblock">IEEE Transactions on Medical Imaging (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib114">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(114)</span>
<span class="ltx_bibblock">
Yao, Y., Liu, F., Zhou, Z., Wang, Y., Shen, W., Yuille, A., Lu, Y.: Unsupervised domain adaptation through shape modeling for medical image segmentation.

</span>
<span class="ltx_bibblock">arXiv preprint arXiv:2207.02529 (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib115">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(115)</span>
<span class="ltx_bibblock">
Yee, P.P., Li, W.: Tumor necrosis: A synergistic consequence of metabolic stress and inflammation.

</span>
<span class="ltx_bibblock">Bioessays <span class="ltx_text ltx_font_bold" id="bib.bib115.1.1">43</span>(7), 2100029 (2021)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib116">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(116)</span>
<span class="ltx_bibblock">
Yoon, D., Kong, H.J., Kim, B.S., Cho, W.S., Lee, J.C., Cho, M., Lim, M.H., Yang, S.Y., Lim, S.H., Lee, J., et al.: Colonoscopic image synthesis with generative adversarial network for enhanced detection of sessile serrated lesions using convolutional neural network.

</span>
<span class="ltx_bibblock">Scientific reports <span class="ltx_text ltx_font_bold" id="bib.bib116.1.1">12</span>(1), 261 (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib117">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(117)</span>
<span class="ltx_bibblock">
Yu, B., Zhou, L., Wang, L., Fripp, J., Bourgeat, P.: 3d cgan based cross-modality mr image synthesis for brain tumor segmentation.

</span>
<span class="ltx_bibblock">In: 2018 IEEE 15th international symposium on biomedical imaging (ISBI 2018), pp. 626–630. IEEE (2018)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib118">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(118)</span>
<span class="ltx_bibblock">
Zhang, Y., Li, X., Chen, H., Yuille, A.L., Liu, Y., Zhou, Z.: Continual learning for abdominal multi-organ and tumor segmentation.

</span>
<span class="ltx_bibblock">In: International Conference on Medical Image Computing and Computer-Assisted Intervention, pp. 35–45. Springer (2023).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/ContinualLearning" title="">https://github.com/MrGiovanni/ContinualLearning</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib119">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(119)</span>
<span class="ltx_bibblock">
Zhao, J., Li, D., Kassam, Z., Howey, J., Chong, J., Chen, B., Li, S.: Tripartite-gan: Synthesizing liver contrast-enhanced mri to improve tumor detection.

</span>
<span class="ltx_bibblock">Medical image analysis <span class="ltx_text ltx_font_bold" id="bib.bib119.1.1">63</span>, 101667 (2020)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib120">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(120)</span>
<span class="ltx_bibblock">
Zhou, Z.: Towards annotation-efficient deep learning for computer-aided diagnosis.

</span>
<span class="ltx_bibblock">Ph.D. thesis, Arizona State University (2021).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/Dissertation" title="">https://github.com/MrGiovanni/Dissertation</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib121">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(121)</span>
<span class="ltx_bibblock">
Zhou, Z., Gotway, M.B., Liang, J.: Interpreting medical images.

</span>
<span class="ltx_bibblock">In: Intelligent Systems in Medicine and Health, pp. 343–371. Springer (2022)

</span>
</li>
<li class="ltx_bibitem" id="bib.bib122">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(122)</span>
<span class="ltx_bibblock">
Zhou, Z., Sodha, V., Siddiquee, M.M.R., Feng, R., Tajbakhsh, N., Gotway, M.B., Liang, J.: Models genesis: Generic autodidactic models for 3d medical image analysis.

</span>
<span class="ltx_bibblock">In: International Conference on Medical Image Computing and Computer-Assisted Intervention, pp. 384–393. Springer (2019).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/ModelsGenesis" title="">https://github.com/MrGiovanni/ModelsGenesis</a>
</span>
</li>
<li class="ltx_bibitem" id="bib.bib123">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">(123)</span>
<span class="ltx_bibblock">
Zhu, Z., Kang, M., Yuille, A., Zhou, Z.: Assembling and exploiting large-scale existing labels of common thorax diseases for improved covid-19 classification using chest radiographs.

</span>
<span class="ltx_bibblock">In: Radiological Society of North America (RSNA) (2022).

</span>
<span class="ltx_bibblock">URL <a class="ltx_ref ltx_url ltx_font_typewriter" href="https://github.com/MrGiovanni/LabelAssemble" title="">https://github.com/MrGiovanni/LabelAssemble</a>
</span>
</li>
</ul>
</section>
</article>
</div>
<footer class="ltx_page_footer">
<div class="ltx_page_logo">Generated  on Mon Sep  9 19:51:32 2024 by <a class="ltx_LaTeXML_logo" href="http://dlmf.nist.gov/LaTeXML/"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img alt="Mascot Sammy" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg=="/></a>
</div></footer>
</div>
</body>
</html>
